DOI: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC17578/<br><br><?xml version="1.0" encoding="utf-8"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<!-- AppResources meta begin -->

<!-- AppResources meta end -->
<!-- TemplateResources meta begin -->

<!-- TemplateResources meta end -->
<!-- Logger begin -->

<!-- Logger end -->
<title>A heterozygous mutation of -actin associated with neutrophil dysfunction and recurrent infection</title>
<!-- AppResources external_resources begin -->

<!-- AppResources external_resources end -->
<!-- Page meta begin -->

<!-- Page meta end -->

</head>
<body class="article">
<div class="grid">
<div class="col twelve_col nomargin shadow">
<!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
<div class="sysmessages">
<noscript>
<p class="nojs">
<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	
</p>
</noscript>
</div>
<!--/.sysmessage-->
<div class="wrap">
<div class="page">
<div class="top">
<div class="universal_header" id="universal_header"><div class="myncbi"></div></div>
<div class="header">
<div class="res_logo">
<h1 class="img_logo"></h1>
<div class="NLMLogo">

<br/>

</div>
</div>
<div class="search"><form action="/pmc/" method="get"><div class="search_form"><label class="offscreen_noflow" for="database">Search database</label><div class="nowrap"><label accesskey="/" class="offscreen_noflow" for="term">Search term</label><div class="nowrap"><input autocomplete="off" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" id="term" name="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." type="text" value=""/></div><button class="button_search nowrap" cmd="go" id="search" type="submit">Search</button></div></div></form></div>
</div>
<!--<component id="Page" label="headcontent"/>-->
</div>
<div class="content">
<!-- site messages -->
<div class="container">
<div class="content eight_col col" id="maincontent">
<div class="navlink-box">

</div>
<!-- Journal banner -->
<div class="pmc-page-banner whole_rhythm"><div><img alt="Logo of pnas" src="/corehtml/pmc/pmcgifs/logo-pnas.gif" usemap="#logo-imagemap"/><map id="logo-imagemap" name="logo-imagemap"><area alt="PNAS" coords="306,2,479,64" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=17578&amp;issue-id=500&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" shape="rect" target="pmc_ext" title="PNAS"/><area alt="Info for Authors" coords="59,34,142,50" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=17578&amp;issue-id=500&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" shape="rect" target="pmc_ext" title="Info for Authors"/><area alt="Subscriptions" coords="143,34,200,50" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=17578&amp;issue-id=500&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" shape="rect" target="pmc_ext" title="Subscriptions"/><area alt="About" coords="201,34,239,50" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=17578&amp;issue-id=500&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" shape="rect" target="pmc_ext" title="About"/><area alt="This Article" coords="0,34,57,50" href="//dx.doi.org/10.1073/pnas.96.15.8693" ref="reftype=publisher&amp;article-id=17578&amp;issue-id=500&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" shape="rect" target="pmc_ext" title="This Article"/></map></div> </div>
<!--component id='MainPortlet' label='search-reference'/-->
<!-- Book content -->
<div class="">
<div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div></div><div> </div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"></div><div class="fm-citation-pmid">PMID: </div></div></div><div class="fm-doctopic no_margin">Medical Sciences</div></div><h1 class="content-title">A heterozygous mutation of -actin associated with neutrophil dysfunction and recurrent infection</h1><div class="half_rhythm"><div class="contrib-group fm-author">, , , , , , and  </div><div class="contrib-group aff-tip" style="display:none"><div id="_co_idm140020968170400"><h3 class="no_margin">Hiroyuki Nunoi</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020967465824"><h3 class="no_margin">Tsuyoshi Yamazaki</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020961899088"><h3 class="no_margin">Hiroyuki Tsuchiya</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020939976688"><h3 class="no_margin">Seishi Kato</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020944936000"><h3 class="no_margin">Harry L. Malech</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020962457312"><h3 class="no_margin">Ichiro Matsuda</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div><div id="_co_idm140020967061552"><h3 class="no_margin">Shiro Kanegasaki</h3><p>Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</p><div>Find articles by </div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers">   </div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140020965057456_ai" style="display:none"><div class="fm-affl" id="N0x981b4b8.0x9c2bb20" lang="en">Institute of Medical Science, University of Tokyo, Minatoku, Tokyo 108-8639, Japan; Department of Pediatrics, Kumamoto University Medical School, Kumamoto 860-8556, Japan; Sagami Chemical Research Center, Sagamihara, Kanagawa 229-0012, Japan; and Host Defense in National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD 20892-1886</div><div id="FN151">H.N. and T.Y. contributed equally to this work.</div><div id="FN152">To whom reprint requests should be sent at present address: Department of Pediatrics, Kumamoto University Medical School, 1-1-1, Honjo, Kumamoto 860, Japan. e-mail: .</div><div id="idm140020941760304">Edited by Thomas P. Stossel, Harvard Medical School, Boston, MA, and approved March 31, 1999</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140020965057456_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 1998 Dec 30</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140020965057456_cpl" style="display:none"><div class="fm-copyright half_rhythm">   1999, The National Academy of Sciences</div></div></div><div class="links-box whole_rhythm" id="pmclinksbox"><div class="fm-panel"><div>This article has been  other articles in PMC.</div></div></div></div><div class="sec"></div><div class="tsec sec" id="idm140020963343040" lang="en"><h2 class="head no_bottom_margin" id="idm140020963343040title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last" id="__p4">A human disorder caused by mutation in nonmuscle actin has not been reported. We report here a variant of nonmuscle actin in a female patient with recurrent infections, photosensitivity, and mental retardation. She also had abnormalities in neutrophil chemotaxis, superoxide production, and membrane potential response. Two-dimensional PAGE analysis of proteins from neutrophils and other cell types from this patient demonstrated a unique protein spot migrating at 42 kDa with pI shifted slightly to neutral relative to normal - and -actin. Digestion peptide mapping and Western blotting showed this spot to be an abnormal actin. A full-length cDNA library was constructed by using mRNA from patients cells and cDNA encoding the mutant -actin molecule was identified by an <em>in vitro</em> translation method. Sequencing of the clones demonstrated a G-1174 to A substitution, predicting a glutamic acid-364 to lysine substitution in -actin and eliminating a <em>Hin</em>fI DNase restriction site found in normal -actin sequence. By <em>Hin</em>fI digestion and by sequencing, the mutation in one allele of patients genomic DNA was confirmed. Though no defect in cell-free polymerization of actin was detected, this defect lies in a domain important for binding to profilin and other actin-regulatory molecules. In fact, the mutant actin bound to profilin less efficiently than normal actin did. Heterozygous expression of mutant -actin in neutrophils and other cells of this patient may act in a dominantnegative fashion to adversely affect cellular activities dependent on the function of nonmuscle actin.</p></div></div><div class="tsec sec headless whole_rhythm" id="idm140020967711824"><p class="p p-first" id="__p5">Phagocytes such as neutrophils play profound roles in defense against microbial infection in the way that they migrate, ingest pathogens, generate reactive oxygen species, and release granule contents to phagosomes and to extracellular medium. The mechanism underlying these activities is associated with, and may require, rapid reorganization of the cytoskeleton. This involves a cyclic process that includes polymerization of G-actin to F-actin, crosslinking of the filaments to form the supramolecular assemblies anchored to membranes, and depolymerization of the F-actin to G-actin ().</p><p id="__p6">Various actin-binding proteins are known to regulate the dynamics of actin cytoskeleton. Schleicher <em>et al.</em> () classified these binding proteins into three categories. The first group of proteins, such as profilin and cofilin, are those that bind to monomeric G-actin and thus reversibly remove polymerizable actin from the equilibrium with F-actin. The second group contains the end-binding proteins or capping proteins, such as gelsolin and cap32/34. They inhibit further addition of monomers to actin polymers and keep filaments short. The third group contains proteins that bind along the side of actin filaments and either stabilize the filaments, crosslink filaments to form three-dimensional networks, anchor filaments to membranes, or work as motors. Binding proteins such as tropomyosin, filamin, spectrin, and myosin are examples of this family.</p><p id="__p7">A variety of functional studies of actin and its associated proteins have been performed by using spontaneous or genetically engineered actin mutants in yeast, in amoebae of slime mold <em>Dictyostelium</em>, in the protozoan <em>Tetrahymena</em>, and in the indirect flight muscle of fruit fly <em>Drosophila</em> (). However, in mammalian cells, the only studies of mutant actin that have been reported are from an <em>in vitro</em> model using a chemically transformed human fibroblast cell line (). Although one clinical case was reported in which the patients symptoms were attributed to a defect in actin polymerization in neutrophils (), a specific structural defect in the actin molecule has not been identified.</p><p class="p" id="__p8">We now report here a human disorder caused by mutation in nonmuscle actin. The patient had repeated infections and other symptoms and had impairment of neutrophil functions. Neutrophils and other cells from the patient were found to contain abnormal -actin together with normal -actin. By sequencing the cDNA that encoded the abnormal actin, we found a single nucleotide substitution. The predicted mutation site in the actin molecule is in a binding site for certain actin-associated proteins such as profilin.</p></div><div class="tsec sec" id="__sec1"><h2 class="head no_bottom_margin" id="__sec1title" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><div class="sec sec-first" id="__sec2"><p></p><h4 class="inline" id="__sec2title">Patient History. </h4><p class="p p-first-last" id="__p9">At the time of initiation of these studies, this female patient was 12 years old, with a history of photosensitivity, recurrent stomatitis, and keratoconjunctivitis since age 3, thrombocytopenia (3  10/l) at age 8, and tuberculous pneumonia, recurrent otitis media, iritis, furunculosis, and a polyathralgia with positive rheumatoid factor since age 9. She exhibited moderate intellectual impairment (IQ score: 54) and had short stature (141.5 cm: 1.4 SD). Laboratory studies showed leukopenia, hyper-IgE, and persistent high levels of C-reactive protein. Serum IgG, IgA, IgM, and IgE levels, respectively, were 3,200 mg/dl, 520 mg/dl, 173 mg/dl, and 2,526 units/ml. Serum protein was 8.8 g/dl, where percents of albumin, 1-, 2-, -, and -globulin were 46.4, 4.3, 10.5, 8.5, and 29.8, respectively. Hemolytic complement activity was in normal range. Leukocyte counts at her admission were 35  10 cells/l, where percent of segmented neutrophils, band form neutrophils, lymphocytes, and monocytes, respectively, were 515, 3035, 30, and 20. The band forms were always the most abundant type of leukocyte in her blood samples, and their ratio to other cells did not change after steroid or epinephrine challenge. There was poor influx of leukocytes to skin window test sites. Subsets of lymphocytes were within the normal range, except OK-Ia1-positive cells, where percent of OKT-3-, OKT-4-, OKT-8-, OKT-10-, OK-Ia1-, and Leu-7-positive cells, respectively, were 64.173.9, 19.530.5, 10.217.4, 21.424.8, 54.854.4, and 2.67.3. Their responses to mitogens were normal. Mild anemia was found (437  10 RBC/l; 9.5 g Hb/dl). Continuous thrombocytopenia was detected (810  10 cells/l), with an increase of megakaryocytes in her bone marrow preparation. Otherwise, bone marrow showed normal cellularity, and percents of myeloblasts, myelocytes, metamyelocytes, band forms, segments, lymphocytes, and erythroblasts were 1.5, 2.5, 14.5, 45, 5.5, 13.5, and 13, respectively. Intradermal skin test gave a positive reaction to Candida antigen, purified protein derivitative, phytohemagglutinin, and streptokinasestreptodornase. Poryphirin metabolites and urinary amino acids were all within normal level. Erythema appeared on her arm after &lt;30 sec of exposure of UV-B, though viability of her cultured fibroblast after exposure to UV light was normal. She was euthyroid, and her TSH value was normal. By 15 years of age, she had developed cardiomegaly, hepatomegaly, and hypothyroidism. At that time, she presented with persistent fevers, and, despite intensive therapy, she died from septicemia.</p></div><div class="sec" id="__sec3"><p></p><h4 class="inline" id="__sec3title">Preparation of Neutrophils and Other Cells. </h4><p class="p p-first-last" id="__p10">Neutrophils and mononuclear cells were obtained as described by Boyum (), and platelets were derived by centrifugation at 800  <em>g</em> for 20 minutes from platelet-rich plasma. Biopsy-derived skin fibroblasts were grown in RPMI medium 1640 supplemented with 20% newborn calf serum. EpsteinBarr virus-transformed B cell lines were established from patient blood lymphocytes according to a method described previously ().</p></div><div class="sec" id="__sec4"><p></p><h4 class="inline" id="__sec4title">Neutrophil Functions. </h4><p class="p p-first-last" id="__p11"><em>In vivo</em> and <em>in vitro</em> assessment of phagocyte chemotaxis was determined by Rebucks skin window test () and Boyden chamber assay (), respectively. Neutrophil superoxide generation was measured as superoxide dismutase-inhibitable reduction of ferricytochrome <em>c</em> (). Membrane potential change of the cells (10 cells/ml) was assayed by using Di-O-C5-[3] (Nippon Kankoh Sikiso Kenkyusho, Okayama, Japan) ().</p></div><div class="sec" id="__sec5"><p></p><h4 class="inline" id="__sec5title">Two-Dimensional (2D) PAGE Analysis and Immunoblotting. </h4><p class="p p-first-last" id="__p12">2D PAGE analysis of proteins in cells was performed according to OFarrel (), with a slight modification. In brief, cells (2  10) were solubilized in 400 l of lysis buffer containing 9.5 M urea, 2% NP40, 2% Pharmalyte (1.6% pH 58 and 0.4% pH 310), and 5% mercaptoethanol. Isoelectric focusing was performed at 500 V for 24 hours followed by the 2D SDS/PAGE. Proteins on the gel were stained with 2D Silver Stain II (Dai-ichi Pure Chemicals, Tokyo). For immunoblotting, proteins on the gel were transferred to poly(vinylidene difluoride) membrane and were stained with the anti-actin monoclonal antibody N350 (Amersham Pharmacia).</p></div><div class="sec" id="__sec6"><p></p><h4 class="inline" id="__sec6title">Construction of cDNA Library. </h4><p class="p p-first-last" id="__p13">To construct a full-length cDNA library, the capping method () was used. In brief, mRNAs with Poly(A) tails were extracted from the patients B cell line and were purified by using poly (T) cellulose column. To remove the 5 phosphate residues from degraded mRNAs, the samples were incubated with alkaline phosphatase. Then, the cap of an intact mRNA was removed by a tobacco acid pyrophosphatase treatment. To the generated 5 phosphate group, a chimeric DNA-RNA oligonucleotide linker was ligated by using T4 RNA ligase. The resulting DNA-capped mRNA was annealed with dT-tailed pKA1 vector primer and was used as the template for cDNA synthesis by Super Script II reverse transcriptase (GIBCO/BRL). The product was digested by <em>Eco</em>RI at the <em>Eco</em>RI sites in the linker and in the vector primer I and was ligated with <em>Escherichia coli</em> DNA ligase. The second strand synthesis of the self-ligated product was performed by using RNase H and <em>E. coli</em> DNA pol. This library is designated as pKA1 library.</p></div><div class="sec" id="__sec7"><p></p><h4 class="inline" id="__sec7title">Cloning and Characterization of Actin cDNAs. </h4><p class="p p-first-last" id="__p14">By using the pKA1 library, the patients actin cDNA clones were selected by hybridization with control -actin probe (). The clones containing the <em>Eco</em>RI-<em>Not</em>I inserts with &gt;1.8 kbp were selected and confirmed to contain the start codon by sequencing of 5 region of the inserts. For <em>in vitro</em> translation of the actin cDNAs, TNT T7 coupled reticulocyte lysate system (Promega) was used. The translation products were labeled with [S]methionine (Amersham Pharmacia) and were analyzed with 2D PAGE.</p></div><div class="sec" id="__sec8"><p></p><h4 class="inline" id="__sec8title">Genomic Analysis of -Actin Gene. </h4><p class="p p-first-last" id="__p15">Genomic DNA was isolated from the patients B cell line. Thirty cycles of amplification by PCR using sets of primers shown in Table  (25 pmol/50 l) were performed in thermal cycler PHC-3 (Techne Laboratories, Cambridge, U.K.) under the following conditions; denaturation at 96C for 45 seconds, annealing at 56C for 45 seconds, and extension at 72C for 1 minute. The reaction mixture contained 50 mM TrisHCl (pH 7.5), 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 0.2 mM dNTPs, 50 ng genomic DNA, and 5 units of recombinant <em>Taq</em> Polymerase and the primers. The products were separated with 2% agarose gel electrophoresis and were extracted from the gel by using Geneclean II kit (Funakoshi, Tokyo). The isolated product was treated with 5 units of <em>Hin</em>fI restriction enzyme (Takara, Tokyo) at 37C for 1 hour and was subjected to PAGE using 5% polyacrylamide gel. The PCR product was cloned into pCRII plasmid by using TA cloning kit (Invitrogen, San Diego) and was sequenced.  </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p33">Oligonucleotide primers for cDNA sequencing and genomic amplification andsequencing</p></div><div class="xtable" data-largeobj="" data-largeobj-link-rid="largeobj_idm140020943580640"><table class="rendered small default_table" frame="hsides" rules="groups"><thead><tr><th colspan="1" rowspan="1">No.</th><th colspan="1" rowspan="1">Primer sequence</th><th colspan="1" rowspan="1">Position, bp </th></tr></thead><tbody align="center"><tr><td colspan="3" rowspan="1">DNA sequencing </td></tr><tr><td colspan="1" rowspan="1">s-1</td><td align="left" colspan="1" rowspan="1">TGTAAAACGACGGCCAGT</td><td colspan="1" rowspan="1">pKA1 primer </td></tr><tr><td colspan="1" rowspan="1">s-2</td><td align="left" colspan="1" rowspan="1">ATCGAGCACGGCATCGTCAC</td><td colspan="1" rowspan="1">252271 </td></tr><tr><td colspan="1" rowspan="1">s-3</td><td align="left" colspan="1" rowspan="1">GGACCTGACTGACTACCTCA</td><td colspan="1" rowspan="1">590609 </td></tr><tr><td colspan="1" rowspan="1">s-4</td><td align="left" colspan="1" rowspan="1">TGGAGAAGAGCTACGAG</td><td colspan="1" rowspan="1">748764 </td></tr><tr><td colspan="1" rowspan="1">s-5</td><td align="left" colspan="1" rowspan="1">ACCATGTACCCTGGCATTGC</td><td colspan="1" rowspan="1">951970 </td></tr><tr><td colspan="1" rowspan="1">s-6</td><td align="left" colspan="1" rowspan="1">CAAGTACTCCGTGTGGATCG</td><td colspan="1" rowspan="1">1,0461,065 </td></tr><tr><td colspan="1" rowspan="1">s-7</td><td align="left" colspan="1" rowspan="1">TGACTTAGTTGCGTTACACC</td><td colspan="1" rowspan="1">1,1781,197 </td></tr><tr><td colspan="1" rowspan="1">s-8</td><td align="left" colspan="1" rowspan="1">AACGGTGAAGGTGACAGCAG</td><td colspan="1" rowspan="1">1,3201,339 </td></tr><tr><td colspan="1" rowspan="1">s-9</td><td align="left" colspan="1" rowspan="1">TGCATTGTTACAGGAAGTCC</td><td colspan="1" rowspan="1">1,4331,452 </td></tr><tr><td colspan="1" rowspan="1">s-10</td><td align="left" colspan="1" rowspan="1">AACTTGAGATGTATGAAGGC</td><td colspan="1" rowspan="1">1,6561,675 </td></tr><tr><td colspan="1" rowspan="1">a-1</td><td align="left" colspan="1" rowspan="1">CACATAGGAATCCTTCTGAC</td><td colspan="1" rowspan="1">184203 </td></tr><tr><td colspan="1" rowspan="1">a-2</td><td align="left" colspan="1" rowspan="1">GAAGGTCTCAAACATGATCT</td><td colspan="1" rowspan="1">403422 </td></tr><tr><td colspan="1" rowspan="1">a-3</td><td align="left" colspan="1" rowspan="1">TGAGGTAGTCAGTCAGGTCC</td><td colspan="1" rowspan="1">590609 </td></tr><tr><td colspan="1" rowspan="1">a-4</td><td align="left" colspan="1" rowspan="1">CTCGTAGCTCTTCTCCAGGG</td><td colspan="1" rowspan="1">744763 </td></tr><tr><td colspan="1" rowspan="1">a-5</td><td align="left" colspan="1" rowspan="1">CGGAACCGCTCATTGCCAAT</td><td colspan="1" rowspan="1">789808 </td></tr><tr><td colspan="1" rowspan="1">a-6</td><td align="left" colspan="1" rowspan="1">GCAATGCCAGGGTACATGGT</td><td colspan="1" rowspan="1">951970 </td></tr><tr><td colspan="1" rowspan="1">a-7</td><td align="left" colspan="1" rowspan="1">GGTGTAACGCAACTAAGTC</td><td colspan="1" rowspan="1">1,1791,197 </td></tr><tr><td colspan="1" rowspan="1">a-8</td><td align="left" colspan="1" rowspan="1">GGACTTCCTGTAACAATGC</td><td colspan="1" rowspan="1">1,4341,452 </td></tr><tr><td colspan="1" rowspan="1">a-9</td><td align="left" colspan="1" rowspan="1">CCTTCATACATCTCAAGTTG</td><td colspan="1" rowspan="1">1,6551,674 </td></tr><tr><td colspan="1" rowspan="1">a-10</td><td align="left" colspan="1" rowspan="1">ATTAACCCTCACTAAAGGGA</td><td colspan="1" rowspan="1">T3 primer </td></tr><tr><td colspan="3" rowspan="1">Genomic amplification and sequencing </td></tr><tr><td colspan="1" rowspan="1">S-1</td><td align="left" colspan="1" rowspan="1">CTCCATCATGAAGTGTGACGTG</td><td colspan="1" rowspan="1">2,6002,621 </td></tr><tr><td colspan="1" rowspan="1">S-2</td><td align="left" colspan="1" rowspan="1">ACATCCGCAAAGACCTGTAC</td><td colspan="1" rowspan="1">2,6232,642 </td></tr><tr><td colspan="1" rowspan="1">A-1</td><td align="left" colspan="1" rowspan="1">GTGTAACGCAACTAAGTCAT</td><td colspan="1" rowspan="1">3,0083,027</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140020943580640" style="display: none"></div><div class="tblwrap-foot"><div id="idm140021001391024"><p class="p p-first-last" id="__p34">The position numbers of the full-length cDNA are based on the report by Ponte <em>et al</em>. () and that of genomic DNA by Nakajima-Iijima <em>et al</em>. (). S and s, sense primers; A and a, antisense primers.</p></div></div></div></div><div class="sec" id="__sec9"><p></p><h4 class="inline" id="__sec9title">DNA Sequencing. </h4><p class="p p-first-last" id="__p16">Cycle sequence method was done by using <em>Taq</em> Dye Deoxy Terminator Cycle Sequencing Kit (PerkinElmer), and the sequencing was performed with an autosequencer (ABI model 373A, PerkinElmer) by using primers listed in Table .</p></div><div class="sec sec-last" id="__sec10"><p></p><h4 class="inline" id="__sec10title">Tests for Polymerizing Ability of Mutant Actin and for Binding Ability to Profilin. </h4><p class="p p-first" id="__p17">The patients B cell line (1.0  10) was labeled with 1 mCi [S]methionine in 10 ml of a methionine-free RPMI medium 1640 supplemented with 10% dialyzed FCS at 37C for 12 hours under 5% CO<sub>2</sub>. Actin polymerization study was done according to the method described by Gordon <em>et al.</em> for <em>Acanthamoeba</em> actin ().</p><p class="p p-last" id="__p18">Binding of actin to profilin was performed by using glutathione <em>S</em>-transferase fusion protein with human profilin [glutathione <em>S</em>-transferase (GST)profilin] coupled to glutathione Sepharose 4B beads (Amersham Pharmacia). For expression of the GST-profilin, human profilin I cDNA inserted to the <em>Bam</em>HI site of pGEX-2T () was used. The S-labeled translation products of normal and mutant actin cDNAs and the GST-profilin-beads, both in a buffer containing 40 mM TrisHCl (pH 7.5), 150 mM NaCl, 1 mM ethylendiaminetetraacetic acid, 0.5% Triton X-100, and 1 mM phenylmethanesulfonyl fluoride, were mixed and incubated at 4C for 2 hours. As a control, the GST beads instead of the GST-profilin beads were used. From the beads, normal and mutant actin bond to profilin were recovered and analyzed by SDS/PAGE.</p></div></div><div class="tsec sec" id="__sec11"><h2 class="head no_bottom_margin" id="__sec11title" style="text-transform: uppercase;">RESULTS</h2><div class="sec sec-first" id="__sec12"><p></p><h4 class="inline" id="__sec12title">Functional Analyses of Patients Neutrophils. </h4><p class="p p-first-last" id="__p19">As seen in Fig.  <em>A</em>, <em>B</em>, and <em>C</em>, respectively, the patients neutrophils demonstrated depressed chemotactic responses, reduced stimulated superoxide generation, and weaker formyl methionyl-leucyl-phenylalanine (fMLP)-induced depolarization/repolarization relative to neutrophils from a normal control.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F1" id="F1"><div class="icnblk_cntnt" id="lgnd_F1"><div></div><!--caption a7--><div class="caption"><p id="__p35">Functional defects in neutrophils from the patient. (<em>A</em>) Chemotactic ability assessed by Boyden chamber method toward 10 () or 10 M fMLP () or 10% zymosan-activated serum (). Vertical numbers indicate numbers of cells in a field. We used different concentrations of fMLP because we found previously that concentrations of fMLP necessary for maximum chemotactic ability were different between the cells from cord and adult blood. Chemotactic ability of cord blood neutrophils was 7080% of normal adults at an optimum concentration of fMLP for adult cells. In each concentration, the cells from the patient exhibited significant chemotaxis deficiency even compared with the ability of the cells from cord blood. (<em>B</em>) Maximum rate of superoxide generation assayed on exposure to 2  10 M fMLP (), 2  10 M fMLP plus 2.5 g/ml cytochalasin D (), or 2.5 g/ml cytochalasin D plus 50 g/ml Con A (). Vertical numbers indicate nanomoles of superoxide/minute/10 neutrophils. Open column, patient; closed column, control. (<em>C</em>) Relative change of fluorescence as an indication of depolarization and repolarization of patient (<em>Left</em>) and control neutrophils (<em>Right</em>). Di-O-C5 [3] was used as the lipophilic probe, and 2  10 M fMLP was used as a stimulus (arrows). Neutrophils immediately after isolation (a) and those stored on ice for 1 hour (b) were used.</p></div></div></div></div><div class="sec" id="__sec13"><p></p><h4 class="inline" id="__sec13title">Characterization of Protein Profiles in Patients Neutrophils. </h4><p class="p p-first" id="__p20">The protein profiles in patients neutrophils were compared with those of control () in 2D PAGE. As shown in Fig. , a unique spot that had the same molecular mass as - and -actin molecules (42 kDa) but a slightly shifted pI value to cathode was found in the patients specimen. Such a spot was not found in neutrophils from control and in leukocytes from &gt;100 individuals (ref. ; data not shown) but did exist in leukocytes, platelets, fibroblasts, and the B cell line from the patient (data not shown).  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F2" id="F2"><div class="icnblk_cntnt" id="lgnd_F2"><div></div><!--caption a7--><div class="caption"><p id="__p36">Analysis of proteins from the patients (<em>A</em>) and normal control neutrophils (<em>B</em>) with 2D PAGE. Closed arrows indicate - (left) and -actin (right), and an open arrow indicates a unique spot. Molecular weight markers are indicated on the left.</p></div></div></div><p class="p p-last" id="__p21">The protein in the spot was reacted with a monoclonal anti-actin antibody and its fingerprint peptide graphic analysis after proteolytic digestion demonstrated high correlation with actin (data not shown). From these results, we concluded the patients cells had an abnormal, mutant actin together with normal - and -actin. The amount of the protein expressed in the cells was comparable to that of -actin molecules.</p></div><div class="sec" id="__sec14"><p></p><h4 class="inline" id="__sec14title">Cloning of Actin cDNAs and Their <em>in Vitro</em> Translation Analysis. </h4><p class="p p-first-last" id="__p22">To determine the mutation responsible for generation of this abnormal actin, a full-length cDNA library constructed from the patients B cell line was used. By hybridization with control -actin cDNA and partial sequencing of 5ends, eight clones, each encoding - or -actin full-length cDNA, were selected. To identify which of these 16 clones coded the mutant actin, <em>in vitro</em> translation products of these clones were analyzed by 2D PAGE. As expected, the <em>in vitro</em> translation yielded both acetylated and nonacetylated products (), and the products of five -actin and all eight -actin clones migrated identically with normal - and -actin protein (Fig.  <em>A</em> and <em>B</em>). However, the translation products of three -actin-encoded clones showed a set of spots that had shifted pI value but the same molecular mass as control (Fig. <em>C</em>). DNA sequencing of these three clones demonstrated a mutation consisting of a G-1174-to-A base pair substitution (Fig. ), predicting a glutamic acid 364-to-lysine substitution in the protein product.    </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F3" id="F3"><div class="icnblk_cntnt" id="lgnd_F3"><div></div><!--caption a7--><div class="caption"><p id="__p37"><em>In vitro</em> translation products of actin cDNAs cloned from patients cells and normal control cells. Products of control -actin (<em>A</em>), -actin (<em>B</em>), and the mutant -actin cDNAs together with control -actin molecules (<em>C</em>) were subjected to 2D PAGE. Closed and open arrows indicate products of normal and mutant actin cDNA, respectively. Each set of spots includes both nonacetylated (right) and acetylated actin (left). Molecular weight markers are indicated on the left.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F4" id="F4"><div class="icnblk_cntnt" id="lgnd_F4"><div></div><!--caption a7--><div class="caption"><p id="__p38">Sequences around 1,174 of the actin cDNAs. The sense sequences and predicted amino acid sequences of control (<em>Upper</em>) and patients specimen (<em>Lower</em>) are shown.</p></div></div></div></div><div class="sec" id="__sec15"><p></p><h4 class="inline" id="__sec15title">Analysis of Patients Genomic DNA. </h4><p class="p p-first" id="__p23">To confirm this mutation in the patients actin gene, genomic DNA was extracted from patients B cell line and was subjected to PCR to amplify the fragments predicted to contain the mutation site using the primers shown in Table . By using one set of these primers (S-2/A-1), the PCR fragment generated from control genomic DNA yielded products of 293 and 405 bp. By sequencing, the 293-bp product was found to be of a pseudo-actin gene that lacked intron 5, and the 405-bp product was of the true actin genomic sequence containing a part of exon 5, intron 5, and a part of exon 6, as expected.</p><p class="p p-last" id="__p24">Because a <em>Hin</em>fI DNase restriction site in the normal sequence was eliminated in the mutant sequence, the 405-bp fragment was isolated and digested with <em>Hin</em>fI restriction enzyme. As shown in Fig. , both undigested 405-bp fragment and the 65- and 340-bp digestion fragments were generated from patient genomic DNA whereas only the 65- and 340-bp fragments were generated from normal control DNA. Similar results were obtained by the analysis of PCR product amplified by using another primer set (S-1/A-1). The results indicate that the normal and mutant alleles are present in the patient cells. The substitution of G-2962 with A in exon 6 was confirmed by sequencing the mutant 405-bp PCR product derived from the patient ().  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F5" id="F5"><div class="icnblk_cntnt" id="lgnd_F5"><div></div><!--caption a7--><div class="caption"><p id="__p39"><em>Hin</em>fI digestion profile of PCR-generated, 405-bp fragment of -actin gene from the patient (Pt) and Control (C<sub>13</sub>). Arrows from top to bottom indicate the position of 405-, 340-, and 65-bp fragments in this order. Mobility of each marker is indicated on the right.</p></div></div></div></div><div class="sec sec-last" id="__sec16"><p></p><h4 class="inline" id="__sec16title">Polymerizing Ability of the Mutant Actin and Binding Ability to Profilin. </h4><p class="p p-first" id="__p25">To know whether the mutant actin molecule has polymerizing ability as normal actin, a cytosole fraction from S-labeled patients B cell line was exposed to polymerization conditions (high magnesium and potassium salts) and was centrifuged. A portion of the resulting precipitate was analyzed by 2D PAGE. The other portion was dialyzed to cause depolymerization. These procedures were repeated three times. As shown in Fig. , the ratios of the mutant and normal actin molecules did not change at each cycle of polymerization, suggesting that the mutant actin can polymeized and depolymerize similar to the normal actin.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F6" id="F6"><div class="icnblk_cntnt" id="lgnd_F6"><div></div><!--caption a7--><div class="caption"><p id="__p40">Copolymerization of mutant and normal actin molecules from the patient B cell line. The cytosolic fraction from [S]methionine-labeled patients cell (<em>A</em>) were polymerized and depolymerized as described in the text. The procedures were repeated three times, and, each time after centrifugation, the precipitate was analyzed by 2D PAGE (<em>B</em><em>D</em>). Open arrows indicate the mutant actin.</p></div></div></div><p class="p p-last" id="__p26">Binding ability of mutant actin to human profilin was examined by using GST-profilin beads. As shown in Fig. , efficiency of binding of mutant actin to profilin was significantly lower than that of normal actin.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" co-legend-rid="lgnd_F7" id="F7"><div class="icnblk_cntnt" id="lgnd_F7"><div></div><!--caption a7--><div class="caption"><p id="__p41">Binding of actin to profilin. The S-labeled translation products of normal and mutant actin cDNAs were incubated with the GST-profilin-beads or GST-beads at 4C for 2 hours. Recovered labeled actin from the beads was subjected to SDS/PAGE. The gel was dried and exposed to Fuji imaging plate, and radioactivity of the actin band was determined by using Bas 2000 (Fuji). The values after subtraction of those obtained by using GST-beads (&lt;10%) are plotted. Open circle, control -actin; closed circle, mutant -actin.</p></div></div></div></div></div><div class="tsec sec" id="__sec17"><h2 class="head no_bottom_margin" id="__sec17title" style="text-transform: uppercase;">DISCUSSION</h2><p class="p p-first" id="__p27">In the present study, we showed that the patient suffering from recurrent infections had the mutant -actin in her neutrophils and other cells together with normal -actin. An actin mutation has not been observed previously in vertebrates, except for a mutation induced in a chemically transformed human fibroblast cell line (). By the analysis of the mutant actin cDNA, a substitution of G-1174 to A was found in the coding region of the -actin molecule, and the mutation was confirmed by examining her genomic DNA. This mutation corresponds to the amino acid transition of glutamic acid-364 to lysine (E364K).</p><p id="__p28">The primary structure of actin is highly conserved in various organisms, implying high biological constraint on structure. The mutation site is situated in a region of actin that is highly conserved. An amino acid substitution at the identical site was reported in actin molecules of <em>Saccharomyces cerevisiae</em> (E364A) (), <em>Dictyostelium</em> (E364H) (), and in the indirect flight muscle-specific <em>Act</em>88F actin in <em>Drosophila melanogaster</em> (E364K) (). Like the mutant actin in our patient, the flight muscle mutant actin occurring in the flightless strain of <em>Drosophila</em> was relatively stable and demonstrated normal <em>in vitro</em> polymerization.</p><p id="__p29">Glutamic acid at residue 364 and its vicinity in actin molecule (residues 355375) has been shown to be important for binding several proteins that modify behavior of the molecule. These proteins include profilin, myosin, cofilin, -actinin, and gelsolin (). In particular, the mutation in the flight muscle actin of <em>Drosophila</em> was shown to affect binding of this mutant actin to profilin. We also showed in this paper that the mutation affected binding of the mutant actin to proflilin. Among the actin binding proteins, profilin, which forms a 1:1 complex with actin, has precisely been studied by mutation (), cross-linking (), and atomic models derived from crystallized complex with actin (). Specifically, <em>Acanthamoeba</em> actin residue E364 can be shown to form a chemical cross-link with lysine-115 of <em>Acanthamoeba</em> profilin () and bovine actin residue E364 to form an intermolecular electrostatic interaction with lysine-125 of bovine profilin, which is homologous to lysine-115 of <em>Acanthamoeba</em> profilin in sequence alignment ().</p><p id="__p30">The mutant actin in the indirect flight muscle was antimorphic for flight ability of <em>Drosophila</em> when expressed in a null strain together with wild-type actin and could not form myofibrils when homozygously expressed in the null strain. By analogy, it is likely that mutant actin molecules of the patient in this study also interfere functions of normal -actin molecules in a dominantnegative manner. It is of note that the mutant -actin described in a chemically transformed human fibroblast line () was also present together with normal a -actin molecule and exerted a dominant negative effect on function ().</p><p id="__p31">It is interesting to note that not only functions directly connected to actin such as chemotaxis but also superoxide generating ability in neutrophils were affected by the mutation. EpsteinBarr virus-transformed B cell line from the patient also had a comparable amount of mutant -actin molecules to that of normal -actin and reduced superoxide generating ability (data not shown). The superoxide-generating system in these cells consists of two membrane proteins designated as gp91-<em>phox</em> and p22-<em>phox</em> that form cytochrome b<sub>558</sub> and three cytosolic proteins, namely p47-<em>phox</em>, p67-<em>phox</em> (), and a small GTP binding protein Rac-p21. The system is in a dormant state in resting neutrophils but is activated on stimulation, where p47-<em>phox</em> and other cytosolic proteins translocate to the membrane and associate to the cytochrome () to form a functionally active enzyme complex on the plasma membrane. Nauseef <em>et al.</em> suggested that p47-<em>phox</em> phospholyrated during activation is more effectively transferred to the membrane as a result of binding to actin (). The abnormality in our patients neutrophils would support a role for actin in the activation of the superoxide-generating phagocyte NADPH oxidase.</p><p class="p p-last" id="__p32">Clinical symptoms of the present patient included photosensitivity and intellectual impairment in addition to the recurrent infections resulting from the neutrophil dysfunction. She did not demonstrate more profound defects in other organ systems. This patient died from infection before our subsequent molecular identification of her defect, precluding additional investigation of her clinical phenotype or defects in cells other than her fibroblast line, B cell line, or previously frozen materials. However, our study suggests that directed mutation at the same site in an engineered mouse or other target animal would generate a viable phenotype and an informative model for further study of the function of nonmuscle actin in higher organisms.</p></div><div class="tsec sec" id="idm140020964785968"><h2 class="head no_bottom_margin" id="idm140020964785968title" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p id="__p42">We were indebted to Dr. E. D. Korn (National Heart, Lung and Blood Institute, National Institutes of Health) for his valuable suggestions and gift of purified <em>Acanthamoeba</em> actin, Dr. T. Takenawa (Institute of Medical Science, University of Tokyo) for providing profilin I cDNA inserted to pGEX-2T, Mr. S. Sekine (Sagami Chemical Research Center) for helping us to construct full-length c-DNA, and Drs. T. Oohira, Y. Yanabe, S. Higuchi, I. Akahosi (Department of Pediatrics, Kumamoto University Medical School), and A. Furuse (Department of Pediatrics, National Sanitarium of Beppu) for their support in the patients care. A part of this work was supported by a Grant-in-Aid for Scientific Research from the Ministry of Education, Science, Sports and Culture of Japan.</p></div></div><div class="tsec sec" id="idm140021035849936"><h2 class="head no_bottom_margin" id="idm140021035849936title" style="text-transform: uppercase;">ABBREVIATIONS</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">2D</td><td>two-dimensional</td></tr><tr><td id="G2">fMLP</td><td>formyl methionyl-leucyl-phenylalanine</td></tr><tr><td id="G3">GST</td><td>glutathione <em>S</em>-transferase</td></tr></tbody></table></div></div></div><div class="tsec sec" id="idm140020940094032"><h2 class="head no_bottom_margin" id="idm140020940094032title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140020940093904"></p><p class="p p-first-last" id="__p46">This paper was submitted directly (Track II) to the <em>Proceedings</em> Office.</p></div></div><div class="tsec sec" id="idm140020945051552"><h2 class="head no_bottom_margin" id="idm140020945051552title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. </div><div class="ref-cit-blk half_rhythm" id="B2">2. </div><div class="ref-cit-blk half_rhythm" id="B3">3. </div><div class="ref-cit-blk half_rhythm" id="B4">4. </div><div class="ref-cit-blk half_rhythm" id="B5">5. </div><div class="ref-cit-blk half_rhythm" id="B6">6. </div><div class="ref-cit-blk half_rhythm" id="B7">7. </div><div class="ref-cit-blk half_rhythm" id="B8">8. </div><div class="ref-cit-blk half_rhythm" id="B9">9. </div><div class="ref-cit-blk half_rhythm" id="B10">10. </div><div class="ref-cit-blk half_rhythm" id="B11">11. </div><div class="ref-cit-blk half_rhythm" id="B12">12. </div><div class="ref-cit-blk half_rhythm" id="B13">13. </div><div class="ref-cit-blk half_rhythm" id="B14">14. </div><div class="ref-cit-blk half_rhythm" id="B15">15. </div><div class="ref-cit-blk half_rhythm" id="B16">16. </div><div class="ref-cit-blk half_rhythm" id="B17">17. </div><div class="ref-cit-blk half_rhythm" id="B18">18. </div><div class="ref-cit-blk half_rhythm" id="B19">19. </div><div class="ref-cit-blk half_rhythm" id="B20">20. </div><div class="ref-cit-blk half_rhythm" id="B21">21. </div><div class="ref-cit-blk half_rhythm" id="B22">22. </div><div class="ref-cit-blk half_rhythm" id="B23">23. </div><div class="ref-cit-blk half_rhythm" id="B24">24. </div><div class="ref-cit-blk half_rhythm" id="B25">25. </div><div class="ref-cit-blk half_rhythm" id="B26">26. </div><div class="ref-cit-blk half_rhythm" id="B27">27. </div><div class="ref-cit-blk half_rhythm" id="B28">28. </div><div class="ref-cit-blk half_rhythm" id="B29">29. </div><div class="ref-cit-blk half_rhythm" id="B30">30. </div><div class="ref-cit-blk half_rhythm" id="B31">31. </div><div class="ref-cit-blk half_rhythm" id="B32">32. </div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr/>Articles from  are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
</div>
<!-- Book content -->
</div>
<div class="four_col col last" id="rightcolumn">
<!-- Custom content above discovery portlets -->
<div class="col6">
</div>
<div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2></div></div><div class="share-buttons" xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><h2>Share</h2></div>
<div data-accid="PMC17578" data-aid="17578" data-aiid="17578" data-domain="pnas" data-domainid="2" data-iid="500" data-md5="0acf8130252b7541cef32ca85726f0f2" data-pmid="10411937" id="ajax-portlets"></div>
<!-- Custom content below discovery portlets -->
<div class="col7">
</div>
</div>
</div>
<!-- Custom content after all -->
<div class="col8">
</div>
<div class="col9">
</div>


<div id="body-link-poppers"></div>
</div>
<div class="bottom">
<div id="NCBIFooter_dynamic">


</div>
<div class="footer" id="footer">
<div class="subfooter"> </div>
<div class="offscreen_noflow" id="external-disclaimer">
        External link. Please review our .
    </div>
<div class="contact_info" id="ncbifooter">
<div id="footer-contents-right">
<div id="nlm_thumb_logo">

</div>
<div id="nih_thumb_logo">

</div>
<div id="hhs_thumb_logo">

</div>
<div id="usagov_thumb_logo">

</div>
</div>
<div id="footer-contents-left">
<p class="address vcard">
 

</p>
 | 
</div>
</div>


</div>
</div>
</div>
<!--/.page-->
</div>
<!--/.wrap-->
</div><!-- /.twelve_col -->
</div>
<!-- /.grid -->

<!-- BESelector tab -->
<noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC17578/&amp;ncbi_app=pmc"/></noscript>
<!-- usually for JS scripts at page bottom -->
<!--<component id="PageFixtures" label="styles"></component>-->
<!-- 8A1B6108F71A1351_0743SID /projects/PMC/PMCViewer@4.46 ptpmc102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->
</body>
</html>