DOI: https://www.spandidos-publications.com/10.3892/mmr.2018.8837<br><br><br>A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report<br>You are using an outdated browser. Upgrade your browser today or install Google Chrome Frame to better experience this site.<br>Molecular MedicineReports<br>A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report<br>This article is mentioned in:<br>Abstract<br>The ACTG1 gene encodes the cytoskeletal protein -actin, which functions in nonmuscle cells and is abundant in the auditory hair cells of the cochlea. Autosomal dominant missense mutations in ACTG1 are associated with DFNA20/26, a disorder that is typically characterized by postlingual progressive hearing loss. To date, 17 missense mutations in ACTG1 have been reported in 20 families with DFNA20/26. The present study described a small family with autosomal dominant nonsyndromic hearing loss. A novel heterozygous missense mutation, c.94C&gt;T (p. Pro32Ser), in ACTG1 was identified using the TruSight One sequencing panel. Notably, congenital hearing loss in our proband was identified by newborn hearing screening at birth. Insilico predictions of protein structure and function indicate that the p. Pro32Ser mutation may result in conformational changes in  actin. The present study expands the understanding of the phenotypic effects of heterozygous missense mutations in the ACTG1 gene. In specific, the present results emphasize that mutations in ACTG1 result in a diverse spectrum of onset ages, including congenital in addition to postlingual onset.<br>Introduction<br>More than 70% of hereditary hearing loss is<br>nonsyndromic and exhibits extremely high genetic heterogeneity.<br>Nonsyndromic hearing impairment (NSHI) exhibits autosomal recessive<br>inheritance in ~7580% of cases, autosomal dominant inheritance in<br>2025% of cases, and X-linked inheritance in 11.5% of cases<br>(). Mutations in at least 30<br>genes have been identified in patients with autosomal dominant<br>sensory NSHI (ADNSHI).<br> was initially identified as a causative<br>gene for ADNSHI linked to the  locus (MIM #604717)<br>on chromosome 17q25.3 in 2003 (,). The<br> gene encodes a protein called -actin, which is part<br>of the actin protein family. The -actin is a cytoskeletal protein,<br>which makes up the structural framework inside cells. This protein<br>is particularly abundant in the specialized cells of inner ear<br>called hair cells, which are essential for normal hearing. Thus<br> is predicted to be crucial for the shape and function<br>of the stereocilia of the cochlear hair cells (). To<br>date, 17 missense mutations in  have been reported in<br>20 families (; ) (,).<br>The majority of these patients present progressive post-lingual<br>hearing loss with ages of onset ranging between the first and<br>fourth decades (). TableI. ACTG1 variants in autosomal<br>dominant sensorineural hearing impairment and Baraitser-Winter<br>cerebrofrontofacial syndrome 2 (BWCFF2). TableI. ACTG1 variants in autosomal<br>dominant sensorineural hearing impairment and Baraitser-Winter<br>cerebrofrontofacial syndrome 2 (BWCFF2). No. Author, yearPhenotypeAA changeExonAge of onsetN(Refs.)1Present studyDFNA20/26p. Pro32Ser2At birth1 family2Miyagawa , 2015DFNA20/26p. Gly48Arg3First decade1 family()3Verloes ,<br>19932016DFNA20/26p. Arg51Asn3First decade1 family()4Zhu ,<br>2003DFNA20/26p. Thr89Ile3Third decade3 families()5Zhu ,<br>2003; Miyagawa , 2015DFNA20/26p. Lys118Asn3First-third<br>decade1 family(,)6Morin ,<br>2009DFNA20/26p. Lys118Met3Second-third<br>decade1 family()7Liu ,<br>2008DFNA20/26p. Ile122Val4First decade1 family()8Baek ,<br>2012DFNA20/26p. Asp187His4First decade (1 year<br>old)1 family()9Yuan ,<br>2016DFNA20/26p. Lys213Arg4Second decade1 family()10Morin ,<br>2009; Miyagawa , 2015DFNA20/26p. Glu241Lys4First decade1 family(,)11Zhu ,<br>2003DFNA20/26p. Pro264Leu4First-second<br>decade1 family()12Mutai ,<br>2013DFNA20/26p. Gly268Ser4First-fourth<br>decade1 family()13van Wijk , 2003DFNA20/26p. Thr278Ile5First-second<br>decade1 family()14Miyagawa , 2015DFNA20/26p. Leu299Val5Second decade2 families(,)15Park ,<br>2013DFNA20/26p. Met305Thr5Third-fourth<br>decade1 family()16Vona ,<br>2014DFNA20/26p. Met325Lys5At birth1 family()17Zhu ,<br>2003DFNA20/26p. Pro332Ala6Second decade1 family()18Rendtorff , 2006DFNA20/26p. Val370Ala6First-second<br>decade1 family()1Di Donato , 2016BWCFF 2p. Asn12Asp21 patient()2Riviere , 2012BWCFF 2p. Thr120Ile31 patient()3Riviere , 2012; Verloes , 2015BWCFF 2p. Ala135Val42 patients(,)4Poirier , 2015BWCFF 2p. Met153Ile41 patient()5Riviere , 2012BWCFF 2p. Ser155Phe43 patients()6Di Donato , 2016BWCFF 2p. Asp179Tyr41 patient()7Riviere , 2012BWCFF 2p. Thr203Lys41 patient()8Di Donato , 2016; Riviere , 2012BWCFF 2p. Arg254Trp42 patients(,)9Di Donato , 2016; Riviere , 2012BWCFF 2p. Arg256Trp42 patients(,)10Di Donato , 2016BWCFF 2p. Glu334Gln61 patient()11Di Donato , 2016BWCFF 2p. Arg335His61 patient()<br>AA, amino<br>acid.<br>The present study presented a small family<br>exhibiting autosomal dominant, congenital, sensorineural hearing<br>loss with an autosomal dominant inherited novel heterozygous C-to-T<br>transition at nucleotide 94 (p. Pro32Ser) of the  gene<br>identified using the TruSight One sequencing panel. The proband was<br>the first case of congenital hearing loss identified by newborn<br>hearing screening. The present study provided a new insight into<br>the genotype-phenotype correlation for  mutations in<br>ADNSHI.<br><br>Case report<br>A small family with ADNSHI was identified at the<br>Eulji Medical Center (Seoul, Korea). The pedigree chart is<br>illustrated in , and the<br>results of Sanger sequencing are presented in .<br>The Institutional Review Board of Eulji General<br>Hospital in Seoul, Korea (IRB approval no. 2014-06-007-001)<br>approved the use of medical records and 2 ml peripheral blood<br>samples in the present study. Written informed consent for genetic<br>testing and medical photography was obtained from all subjects<br>prior to participation.<br>Patient 1 (proband)<br>A boy was born at term with a birth weight of 2,770<br>g (1025th percentile) and an occipitofrontal circumference of 34.5<br>cm (7590th percentile) as the first child of a non-consanguineous<br>33-year-old Filipina mother and a 49-year-old Korean father<br>(). He had one<br>sibling who was healthy. At birth, he had a complete bilateral<br>cleft lip and palate. He did not pass the initial screening test<br>with otoacoustic emissions at birth. He exhibited bilateral severe<br>hearing loss on auditory brainstem response at threshold 90 dB<br>above normal adult hearing level (nHL; ) and auditory steady-state<br>response audiometry at threshold 80 dB nHL (). He underwent surgical repair of<br>the cleft lip (cheiloplasty) at 4 months and cleft palate<br>(palatoplasty) at 12 months. He received cochlear implant (CI)<br>surgery in the right ear at 22 months and in the left ear at 4<br>years. When he visited our clinic (Nowon Eulji Medical Center,<br>Eulji University, Seoul, Republic of Korea) at 4 years and 6 months<br>old, his body weight was 17.8 kg [+0.00 standard deviation (SD)],<br>height was 99.9 cm (1.53 SD), and head circumference was 53 cm<br>(+1.44 SD). On examination, he presented a distinctive face owing<br>to the reconstructed philtrum and upper lip by cheiloplasty<br>(). He had no coloboma or<br>ptosis in an eye examination performed by an ophthalmologist. There<br>were no visible abnormalities of the external ear. There were no<br>skeletal abnormalities. He exhibited no seizures, movement<br>problems, or other behavioral problems. He revealed developmental<br>language delay based on the sequenced language scale of infants at<br>4 years and 5 months of age; expressive language function was at<br>the 21-month-old level, and receptive language function was at the<br>23-month-old level. This was attributed to congenital deafness with<br>relatively delayed cochlear implants. The results of brain magnetic<br>resonance imaging (MRI) were normal. There were no associated<br>structural ear abnormalities in temporal bone computer tomography<br>findings. Electrocardiography and chest radiography results were<br>normal. Echocardiograms exhibited a very small patent ductus<br>arteriosus. No evidence of malformation of the kidneys was observed<br>in an abdominal ultrasound evaluation. The laboratory tests,<br>including a complete blood count, chemistry panel, lipid profile,<br>thyroid function test, and urinalysis, were all normal.<br><br>Patient 2 (father)<br>The patient's father first visited our clinic at 53<br>years of age with his son. He was the second child of healthy<br>non-consanguineous Korean parents. There was no family history of<br>neurologic disease, deafness, or developmental delay, with the<br>exception of his son. At this visit, he exhibited profound<br>pre-lingual deafness (). He<br>did not acquire speech and language. On examination, he had no<br>specific dysmorphic craniofacial features. Brain MRI was<br>normal.<br><br>Results<br>Cytogenetic analysis<br>To identify the causative genetic alterations in the<br>proband (III-1), cytogenetic and analysis were performed.<br>Peripheral blood lymphocytes were cultured for 72 h according to<br>routine cytogenetic protocols. G-banding normal chromosomes (46,<br>XY) was found in the proband. A genomic microarray of DNA from the<br>proband's peripheral blood using a 750K High-resolution Genotyping<br>SNP Microarray (Affymetrix; Thermo Fisher Scientific, Inc.,<br>Waltham, MA, USA) revealed no abnormalities.<br><br>Mutation analysis<br>Considering the genetic heterogeneity of the disease<br>and for differential diagnosis, clinical sequencing of regions<br>containing Mendelian disease-causing genes in the proband (III-1)<br>was performed for genetic confirmation. Genomic DNA was extracted<br>from peripheral blood leukocytes using the Wizard Genomic DNA<br>Purification kit following the manufacturer's protocols (Promega<br>Corporation, Madison, WI, USA). Library preparation was conducted<br>using TruSight One Sequencing Panel (Illumina, Inc., San Diego, CA,<br>USA), which included 125,395 probes targeting a 12-Mb region<br>spanning 4,813 genes. Massively parallel sequencing was performed<br>using Illumina MiSeq platform (Illumina, Inc.). Sequence reads were<br>aligned to hg19 using Burrow-Wheeler Aligner (version 0.7.12).<br>Duplicate reads were removed by Picard (version 1.96, ). Local realignment and<br>base quality recalibration were conducted using Genome Analysis<br>Toolkit (GATK version 3.5). Variant calling was performed using<br>GATK HaplotypeCaller. Variants were annotated using Variant Effect<br>Predictor and dbNSFP (version 3.0b2a). Sanger sequencing was used<br>to determine the genotypes of affected and unaffected family<br>members at candidate loci. From sequencing data of the proband with<br>the TruSight One Panel, it was expected to discover a heterozygote<br>ADNSHI-causing mutation in this family. Mutations in known genes<br>causing autosomal recessive in addition to autosomal dominant<br>syndromic/nonsyndromic hearing loss were also evaluated in the<br>exome data of the proband. Two missense mutations were detected:<br>c.94C&gt;T (p. Pro32Ser) in  and c.94C&gt;T (p. Pro32Ser)<br>in . The mutation c.94C&gt;T (p. Pro32Ser) in<br> displayed lack of segregation in affected and<br>unaffected members of this family. The mutation was not observed in<br>another affected family member (II-2) and was observed in two<br>unaffected family members (II-3 and III-2). In contrast, the<br>heterozygous novel missense mutation, c.94C&gt;T (p. Pro32Ser), in<br>the  gene was supported as a disease-causing mutation<br>based on segregation in the family. Sanger sequencing confirmed<br>that the affected family members (III-1 and II-2) were heterozygous<br>for the mutation, while the mutation was not observed in the<br>unaffected family members (I-2, II-3 and III-2; ). This mutation is absent from<br>controls in NHLBI GO Exome Sequencing Project (), 1000 Genomes<br>Project Phase 3 (), Exome Aggregation<br>Consortium v0.3 (), and Korean Reference<br>Genome Database ().<br>Multiple lines of evidence from  prediction<br>indicated that the mutation was potentially deleterious. The<br>characteristics of the novel missense mutation are summarized in<br>. TableII. Summary of the disease-causing novel<br>mutation in ACTG1. TableII. Summary of the disease-causing novel<br>mutation in ACTG1. Gene(Refs.)Position<br>(hg19)chr17q25<br>(79,479,287 base pair)Exome2/7Accession<br>numberNM_001614.3Nucleotide<br>changec.94C&gt;T,<br>heteroAmino acid<br>changep. Pro32SerHGMDNone()Classification by<br>ACMG guidelinesLikely<br>pathogenic()Allele frequency in<br>controlsdbSNP<br>142No rsID1000Gp3NoneESP6500NoneExACNoneKRGdbNoneMissense<br>predictionSIFT_<br>predDeleteriousLRT_<br>predUnknownPolyPhen2 HDIV_ predProbably<br>damagingPolyphen2_HVAR_ predProbably<br>damagingMutationTaster_predDisease<br>causingMutationAssessor_predMedium predicted<br>functionalMetaSVM_predDeleteriousMetaLR_predDeleteriousFATHMM_predDeleteriousPROVEAN_predDamagingCADD_phred16.96Nucleotide<br>conservation predictionGEFF++_RS3.99PhyloP7way_vertebrate0.917phastCons7way_vertebrate0.98SiPhy_29way_logOdds15.0358<br>In addition, genes known to serve a role in<br>craniofacial development including cleft lip and palate were<br>examined. A missense mutation c.251A&gt;T (p. Glu84Val) in the<br> gene was identified but lacked segregation in this<br>family, as the mutation was also observed in two unaffected family<br>members (II-3 and III-2).<br><br>Structural modelling of<br>p. Pro32Ser<br>The effect of the p. Pro32Ser mutation on the<br>structure and function of the  protein product,<br>-actin, was next assessed. The p. Pro32Ser region of <br>was checked against the protein feature view of the Protein Data<br>Bank () entries mapped to<br>a UniProt Knowledgebase () sequence. The native amino<br>acid at position 32 is hydrophobic and in a predicted potentially<br>disordered region (). The<br>substitution of proline (nonpolar, hydrophobic) by serine<br>(uncharged polar, hydrophilic) is likely to change the<br>hydrophobicity of the hydrophobic region and influence hydrophobic<br>interactions. It may affect protein folding or decrease the<br>stability of the native protein structure of -actin. Pro32Ser is<br>also close to a phosphorylation site (). Serine is one of most commonly<br>phosphorylated amino acids. A substitution by serine could affect<br>protein phosphorylation and post-translational modifications.  predictions were performed to infer the<br>three-dimensional structure of -actin using UCSF chimera<br>(). The<br>32. Pro in wild-type -actin participates in two hydrogen bonds<br>(distances, 2.804 A and 2.864 A), as demonstrated in . Structural modeling of mutant<br>32. Ser identified a new hydrogen bond (distance, 3.367 A) that<br>interacted with 15. Gly ().<br>Hydrogen bonding serves a particularly important role in secondary<br>structure formation and the determination of three-dimensional<br>structures. Prolines are flexible amino acids and are often in <br>turns with glycine. In a  turn, there is a tight 180 reversal in<br>the direction of the polypeptide chain, and proline can readily<br>assume the cis configuration, which facilitates a tight turn.<br>Accordingly, the Pro32Ser mutation may result in a conformational<br>change in -actin. Molecular modelling analyses therefore predicted<br>alterations in -actin and actin-based structures following this<br>mutation.<br><br>Discussion<br>The auditory hair cells are located within the<br>spiral organ of Corti on the thin basilar membrane in the cochlea<br>of the inner ear. The inner hair cells transform the sound<br>vibrations in the fluids of the cochlea into electrical signals<br>that are then relayed via the auditory nerve to the auditory<br>brainstem and to the auditory cortex. Humans and other mammals are<br>generally incapable of regrowth of the inner ear cells that convert<br>sound into neural signals when those cells are damaged by age or<br>disease. Actins are highly conserved proteins involved in cell<br>motility and cytoskeletal maintenance. -Actin is encoded by the<br> gene and is identified in the cytoplasm of non-muscle<br>cells. It is abundant in the auditory hair cells of the cochlea.<br>Actin filaments are essential for the shape and function of the<br>stereocilia of hair cells. An alteration in actin filament<br>regulation caused by actin-binding proteins is a major factor in<br>deafness caused by  mutations (). The DFNA20/26 phenotype includes<br>hearing loss with post-lingual onset and an increasing degree of<br>hearing loss with age (DFNA20/26, MIM# 604717). Impaired actin<br>structures may be more susceptible to age-dependent degeneration,<br>resulting in progressive late-onset, post-lingual hearing loss. In<br>patients with very early-onset hearing loss, suspected hearing loss<br>is rarely reported (). The<br>proband in the present study was the first case of congenital<br>hearing loss to be identified by a newborn hearing screening test.<br>His father exhibited pre-lingual profound hearing impairment. It is<br>hypothesised that congenital and pre-lingual-onset hearing loss is<br>possible in patients with ADNSHI associated with <br>mutations.<br>A missense mutation in  has also been<br>identified in patients with Baraitser-Winter cerebrofrontofacial<br>syndrome 2 (BWCFF2, MIM# 614583), which is a multiple congenital<br>malformation syndrome including specific facial gestalt, short<br>status, brain malformation, and sensorineural hearing loss<br>(). Eleven mutations in 16<br>patients have been detected in BWCFF2 to date (; ). All mutations in BWCFF2, in<br>addition to DFNA20/26, are heterozygous missense mutations. The<br>spectrum of pathogenic mutations observed in DFNA20/26 does not<br>overlap with the spectrum of pathogenic mutations observed in<br>BWCFF2 (). Mutations<br>causing DFNA20/26 and BWCFF2 are distributed over the entire exome<br>without regional clusters, as shown in . The molecular genetic mechanisms<br>of  heterozygous missense mutations in DFNA20/26 and<br>BWCFF2 are unclear; this is an important open question for future<br>research. Up to 83% of patients with BWCFF2 have congenital hearing<br>loss. Riviere  ()<br>suggested that BWCFF2 represents the severe end of a spectrum of<br>cytoplasmic actin-associated phenotypes that begins with ADNSHI and<br>extends to BWCFF2. In the present study, the father (II-2) carried<br>a  mutation in  and presented with<br>non-syndromic pre-lingual deafness. The proband (III-1) had<br>additional features of cleft lip and cleft palate. While cleft lip<br>and cleft palate may be identified in various craniofacial<br>syndromes, environmental influences may also cause or interact with<br>genetics to produce orofacial clefting. In addition, the proband<br>exhibited no other minor or major anomalies suggestive of BWCFF2<br>with the exception of cleft lip and palate.<br>Advances in the acquisition of clinical and<br>molecular data may improve molecular genetic diagnoses, including<br>those based on  mutations, and may contribute to<br>therapeutic decisions. Previous studies have suggested that CI<br>surgery is a good therapeutic option for patients with <br>mutations, as the etiology involves the cochlea and normal brain<br>structures; therefore, comparatively improved outcomes are<br>predicted (,). In the present study, the proband<br>with congenital deafness underwent CI surgery. Although cochlear<br>implantation was performed late, at 2 and 4 years old, he received<br>language therapy and has the ability to communicate. The present<br>findings support the recommendation for early CI surgery in<br>patients with  mutations.<br>In summary, a novel heterozygous missense mutation<br>P32S in the  gene was identified in a small family with<br>autosomal dominant nonsyndromic hearing loss. The present findings<br>expand our understanding of the phenotypes associated with<br>. Specifically, the results of the present study<br>emphasized that mutations in  result in a diverse<br>spectrum of onset ages, including congenital in addition to<br>post-lingual onset.<br><br>Acknowledgements<br>This research was supported by the Basic Science<br>Research Program through the National Research Foundation of Korea<br>funded by the Ministry of Science, ICT and Future Planning (grant<br>no. 2014R1A1A1007569).<br>References<br>Smith RJH, Shearer AE, Hildebrand MS and<br>Van Camp G: Deafness and hereditary hearing loss<br>overviewGeneReviews. Pagon RA, Adam MP, Ardinger HH, et al:<br>University of Washington; Seattle: 19932016<br>Zhu M, Yang T, Wei S, DeWan AT, Morell RJ,<br>Elfenbein JL, Fisher RA, Leal SM, Smith RJ and Friderici KH:<br>Mutations in the gamma-actin gene (ACTG1) are associated with<br>dominant progressive deafness (DFNA20/26). Am J Hum Genet.<br>73:10821091. 2003.  :  :<br>van Wijk E, Krieger E, Kemperman MH, De<br>Leenheer EM, Huygen PL, Cremers CW, Cremers FP and Kremer H: A<br>mutation in the gamma actin 1 (ACTG1) gene causes autosomal<br>dominant hearing loss (DFNA20/26). J Med Genet. 40:879884. 2003.<br> :  :<br>Morin M, Bryan KE, Mayo-Merino F, Goodyear<br>R, Mencia A, Modamio-Hybjr S, del Castillo I, Cabalka JM,<br>Richardson G, Moreno F, et al: In vivo and in vitro effects of two<br>novel gamma-actin (ACTG1) mutations that cause DFNA20/26 hearing<br>impairment. Hum Mol Genet. 18:30753089. 2009.  :  :<br>Yuan Y, Gao X, Huang B, Lu J, Wang G, Lin<br>X, Qu Y and Dai P: Phenotypic Heterogeneity in a DFNA20/26 family<br>segregating a novel ACTG1 mutation. BMC Genet. 17:332016.<br> :  :<br>Bryan KE, Wen KK, Zhu M, Rendtorff ND,<br>Feldkamp M, Tranebjaerg L, Friderici KH and Rubenstein PA: Effects<br>of human deafness gamma-actin mutations (DFNA20/26) on actin<br>function. J Biol Chem. 281:2012920139. 2006.  :  :<br>Miyagawa M, Nishio SY, Ichinose A, Iwasaki<br>S, Murata T, Kitajiri S and Usami S: Mutational spectrum and<br>clinical features of patients with ACTG1 mutations identified by<br>massively parallel DNA sequencing. Ann Otol Rhinol Laryngol. 124<br>Suppl 1:84S93S. 2015.  :  :<br>Liu P, Li H, Ren X, Mao H, Zhu Q, Zhu Z,<br>Yang R, Yuan W, Liu J, Wang Q and Liu M: Novel ACTG1 mutation<br>causing autosomal dominant non-syndromic hearing impairment in a<br>chinese family. J Genet Genomics. 35:553558. 2008.  :  :<br>Baek JI, Oh SK, Kim DB, Choi SY, Kim UK,<br>Lee KY and Lee SH: Targeted massive parallel sequencing: The<br>effective detection of novel causative mutations associated with<br>hearing loss in small families. Orphanet J Rare Dis. 7:602012.<br> :  :<br>Mutai H, Suzuki N, Shimizu A, Torii C,<br>Namba K, Morimoto N, Kudoh J, Kaga K, Kosaki K and Matsunaga T:<br>Diverse spectrum of rare deafness genes underlies early-childhood<br>hearing loss in Japanese patients: A cross-sectional, multi-center<br>next-generation sequencing study. Orphanet J Rare Dis. 8:1722013.<br> :  :<br>Miyagawa M, Nishio SY, Ikeda T, Fukushima<br>K and Usami S: Massively parallel DNA sequencing successfully<br>identifies new causative mutations in deafness genes in patients<br>with cochlear implantation and EAS. PLoS One. 8:e757932013.<br> :  :<br>Park G, Gim J, Kim AR, Han KH, Kim HS, Oh<br>SH, Park T, Park WY and Choi BY: Multiphasic analysis of whole<br>exome sequencing data identifies a novel mutation of ACTG1 in a<br>nonsyndromic hearing loss family. BMC Genomics. 14:1912013.<br> :  :<br>Vona B, Muller T, Nanda I, Neuner C,<br>Hofrichter MA, Schroder J, Bartsch O, Laig A, Keilmann A, Schraven<br>S, et al: Targeted next-generation sequencing of deafness genes in<br>hearing-impaired individuals uncovers informative mutations. Genet<br>Med. 16:945953. 2014.  :  :<br>Verloes A, Drunat S, Pilz D and Di Donato<br>N: Baraitser-Winter cerebrofrontofacial syndromeGeneReviews. Pagon<br>RA, Adam MP, Ardinger HH, et al: University of Washington; Seattle:<br>19932016<br>Rendtorff ND, Zhu M, Fagerheim T, Antal<br>TL, Jones M, Teslovich TM, Gillanders EM, Barmada M, Teig E, Trent<br>JM, et al: A novel missense mutation in ACTG1 causes dominant<br>deafness in a Norwegian DFNA20/26 family, but ACTG1 mutations are<br>not frequent among families with hereditary hearing impairment. Eur<br>J Hum Genet. 14:10971105. 2006.  :  :<br>Di Donato N, Kuechler A, Velgano S,<br>Heinritz W, Bodurtha J, Merchant SR, Breningstall G, Ladda R, Sell<br>S, Altmuller J, et al: Update on the ACTG1-associated<br>Baraitser-Winter cerebrofrontofacial syndrome. Am J Med Genet A.<br>170:26442651. 2016.  :  :<br>Riviere JB, van Bon BW, Hoischen A,<br>Kholmanskikh SS, O'Roak BJ, Gilissen C, Gijsen S, Sullivan CT,<br>Christian SL, Abdul-Rahman OA, et al: De novo mutations in the<br>actin genes ACTB and ACTG1 cause Baraitser-Winter syndrome. Nat<br>Genet. 44:440444. 2012.  :  :<br>Verloes A, Di Donato N, Masliah-Planchon<br>J, Jongmans M, Abdul-Raman OA, Albrecht B, Allanson J, Brunner H,<br>Bertola D, Chassaing N, David A, et al: Baraitser-Winter<br>cerebrofrontofacial syndrome: Delineation of the spectrum in 42<br>cases. Eur J Hum Genet. 23:292301. 2015.  :  :<br>Poirier K, Martinovic J, Laquerriere A,<br>Cavallin M, Fallet-Bianco C, Desguerre I, Valence S,<br>Grande-Goburghun J, Francannet C, Deleuze JF, et al: Rare ACTG1<br>variants in fetal microlissencephaly. Eur J Med Genet. 58:416418.<br>2015.  :  :<br>Human Gene Mutation Database Professional.<br>version 2017.1.<br>Richards S, Aziz N, Bale S, Bick D, Das S,<br>Gastier-Foster J, Grody WW, Hegde M, Lyon E, Spector E, et al:<br>Standards and guidelines for the interpretation of sequence<br>variants: A joint consensus recommendation of the American college<br>of medical genetics and genomics and the association for molecular<br>pathology. Genet Med. 17:405424. 2015.  :  :<br>Related Articles<br>Print ISSN:<br>Online ISSN:<br>This site uses cookies<br>You can change your cookie settings at any time by following the instructions in our . To find out more, you may read our<br>.<br>0<br>)--><br>Copy and paste a formatted citation<br>Spandidos Publications style<br>Lee C, Jang J and Jin HS: A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report. Mol Med Rep 17: 7611-7617, 2018<br>APA<br>Lee, C., Jang, J., &amp; Jin, H. (2018). A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report. Molecular Medicine Reports, 17, 7611-7617. https://doi.org/10.3892/mmr.2018.8837<br>MLA<br>Lee, C.,  Jang, J.,  Jin, H."A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report". Molecular Medicine Reports 17.6 (2018): 7611-7617.<br>Chicago<br>Lee, C.,  Jang, J.,  Jin, H."A novel missense mutation in the ACTG1 gene in a family with congenital autosomal dominant deafness: A case report". Molecular Medicine Reports 17, no. 6 (2018): 7611-7617. https://doi.org/10.3892/mmr.2018.8837<br>