<html><body><table class="support-info__table table article-section__table"> <thead> <th id="article-filename" scope="col">Filename</th> <th id="article-description" scope="col">Description</th> </thead> <tbody> <tr id=""> <td headers="article-filename"><a href="/action/downloadSupplement?doi=10.1002%2Fhumu.23205&amp:amp:file=humu23205-sup-0001-SuppMat.docx">humu23205-sup-0001-SuppMat.docx</a>17.4 MB                            </td> <td headers="article-description"><p><b>Supp. Figure S1</b>: Result of whole genome parametric (Merlin) linkage analysis for family 1.</p> <p><b>Supp. Figure S2</b>: Electropherograms from Sanger confirmation in members of family 1, (A) showing <i>SARS</i>: c.514G&amp:gt:A, and (B) <i>ZNF697</i>: c.472C&amp:gt:A, heterozygous and homozygous sequences.</p> <p><b>Supp. Figure S3</b>: <i>SARS</i> expression (readcounts) determined by RNA‐sequencing of various  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: Table File 1" >brain</span>  tissues.</p> <p><b>Supp. Figure S4</b>: SARS active site. (A) Residues p.His170 (H170, dark blue) and F316/383 (orange) stabilize a β‐sheet of the aminoacylation domain. p.Asp172 (D172, red) faces away from the aminoacylation core, but is located directly adjacent to the N‐terminus of the UNE‐S domain (yellow). Light blue: ATP binding sites: pink: serine binding sites. Orientation of (B) wild type p.Asp172 (D172, green) and (C) mutant p.Asn172 (N172, red). The nuclear localization signal (NLS) is located seven amino acids downstream of p.Pro475 (P475, yellow).</p> <p><b>Supp. Figure S5</b>: WARS2 expression (readcounts) determined by RNA‐sequencing of various  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: Table File 1" >brain</span>  tissues.</p> <p><b>Supp. Figure S6</b>: Electropherograms from Sanger confirmation in members of family 2, showing <i>WARS2</i>: c.37T&amp:gt:G and c325delA sequences. Father and mother show heterozygous sequences for the frameshift and the missense change, respectively.The patients show heterozygous sequences for both mutations.</p> <p><b>Supp. Figure S7</b>: EYFP‐WARS2 wild type and mutant localize in the mitochondria (Control of mitochondrial fractionation). Protein lysate from mitochondrial fraction (MT), cytosolic fraction (CF) from HEK293T cells transfected either with EYFP‐WARS2 wild type (WT) or mutant (p.Trp13Gly) were run on SDS‐PAGE. The gel was blotted and probed with an α‐GFP antibody. The blot was subsequently probed with α‐tubulin and α‐PHB1 (mitochondrial marker) antibodies as loading controls for cytosolic and mitochondrial fractions, respectively.</p> <p><b>Supp. Figure S8</b>: Immunofluorescence analysis, using a specific antibody against SDHA on cells overexpressing EYFP‐WARS2 wild type (WT), compared to cells overexpressing the mutant protein (p.Trp13Gly) and control. In the cells overexpressing WT‐WARS2 (cells with green signal, upper panel) a strong reduction of endogenous SDHA signal is visible (red signal in the corresponding cells, upper panel) in comparison to those expressing p.Trp13Gly.</p> <p><b>Supp. Table S1</b>: Primers for amplification of <i>SARS</i> and <i>WARS2</i> genes.</p> <p><b>Supp. Table S2</b>: Primers for mutagenesis and cloning of <i>SARS</i> and <i>WARS2</i> in different expression vectors.</p> <p><b>Supp. Table S3</b>: Variants identified by whole‐exome sequencing in one affected (V:7) from family 2.</p> </td> </tr> </tbody> </table></body></html>