DOI: https://onlinelibrary.wiley.com/doi/full/10.1002/humu.23205<br><br><br>Mutations of the aminoacyl tRNA synthetases SARS and WARS2 are implicated in the etiology of autosomal recessive  <span style="color:red" title="HPO: HP:0001249
CONF: 0.9938634
PAT: None
NEG: False
REL: False
SECTION: Unassigned" >intellectual disability</span>  - Musante - 2017 - Human Mutation - Wiley Online Library<br>Working off-campus? Learn about our remote access options<br>By continuing to browse this site, you agree to its use of cookies as described in our Cookie Policy.x<br>Access ByUtrecht University<br>Access byUtrecht University<br><h5><br> Search within<br>Login / Register<br>Human MutationVolume 38, Issue 6<br> <br> RESEARCH ARTICLE<br>Full Access<br> <h1>Mutations of the aminoacyl tRNA synthetases SARS and WARS2 are implicated in the etiology of autosomal recessive  <span style="color:red" title="HPO: HP:0001249
CONF: 0.9938634
PAT: None
NEG: False
REL: True
SECTION: Mutations of the aminoacyl tRNA synthetases SARS and WARS2 are implicated in the etiology of autosomal recessive intellectual disability" >intellectual disability</span> </h1> <br>First published: 25 February 2017<br> https://doi-org.proxy.library.uu.nl/10.1002/humu.23205Citations: 31<br><h5></h5> <br> <h5></h5> <br> <h5></h5> <br> <h5></h5> <br> <h5><br> <br> Contract Grant Sponsors: Iranian National Science Foundation: Max Planck Innovation Fund, GENCODYS.<br><h5><br> <br> Communicated by Christine Van Broeckhoven<br>AboutSectionsPDF<br> Tools<br> ShareShare<br> Give access<br> Share full text access<h5>Share full-text access<br> Please review our Terms and Conditions of Use and check box below to share full-text version of article. I have read and accept the Wiley Online Library Terms and Conditions of UseShareable LinkUse the link below to share a full-text version of this article with your friends and colleagues. Learn more.<br> Copy URL<br> Share a link<br> Share on<br><h2>Abstract</h2> <br> <br>  <span style="color:red" title="HPO: HP:0001249
CONF: 0.9938634
PAT: None
NEG: False
REL: True
SECTION: Abstract" >Intellectual disability</span>  (ID) is the hallmark of an extremely heterogeneous group of disorders that comprises a wide variety of syndromic and non syndromic phenotypes. Here, we report on mutations in two aminoacyl tRNA synthetases that are associated with ID in two unrelated Iranian families. In the first family, we identified a homozygous missense mutation (c.514G&gt:A, p. Asp172Asn) in the cytoplasmic seryl tRNA synthetase () gene. The mutation affects the enzymatic core domain of the protein and impairs its enzymatic activity, probably leading to reduced cytoplasmic tRNA concentrations. The mutant protein was predicted to be unstable, which could be substantiated by investigating ectopic mutant SARS in transfected HEK293T cells. In the second family, we found a compound heterozygous genotype of the mitochondrial tryptophanyl tRNA synthetase () gene, comprising a nonsense mutation (c.325delA, p. Ser109Alafs*15), which very likely entails nonsense mediated mRNA decay and a missense mutation (c.37T&gt:G, p. Trp13Gly). The latter affects the mitochondrial localization signal of WARS2, causing protein mislocalization. Including , which we have recently implicated in the etiology of ID, three genes with a role in tRNA aminoacylation are now associated with this condition. We therefore suggest that the functional integrity of tRNAs in general is an important factor in the development and maintenance of human  <span style="color:red" title="HPO: HP:0100543
CONF: 0.9595157
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >cognitive</span>  functions.<br><h2>1 INTRODUCTION</h2> <br> With a prevalence between 1% and 3%,  <span style="color:red" title="HPO: HP:0001249
CONF: 0.9938634
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >intellectual disability</span>  (ID) is among the most important problems in healthcare. Particularly, hereditary autosomal recessive forms of the disorder (autosomal recessive  <span style="color:red" title="HPO: HP:0001249
CONF: 0.9938634
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >intellectual disability</span>  [ARID]) have a very heterogeneous molecular basis, and genes with an increased number of disease causing mutations are not common (for review see, e.g., Khan et al., 2016: Musante &amp: Ropers, 2013). Interestingly, apart from genes specifically expressed in synapses or in the nervous system, many ubiquitously expressed housekeeping genes have been implicated in ARID and neurological disorders (Najmabadi et al., 2011). In line with these findings, we identified mutations in two aminoacyl tRNA synthetases (ARSs) in families with ARID. ARSs are essential and ubiquitously expressed enzymes and responsible for ligating amino acids to cognate tRNA molecules (aminoacylation) in mitochondria and in the cytosol. Mutations in at least five mitochondrial ARSs have already been associated with  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >brain</span>  specific phenotypes, and at least four genes encoding cytoplasmic ARS have been implicated in inherited  <span style="color:red" title="HPO: HP:0003477
CONF: 0.9488188
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >peripheral neuropathy with an axonal</span>  pathology (for review see, e.g., Oprescu, Griffin, Beg, &amp: Antonellis, 2016: Wallen &amp: Antonellis, 2013: Yao &amp: Fox, 2013).<br>Here, we report on two Iranian families affected by ARID with similar accompanying features (Table 1). Using homozygosity mapping, next generation sequencing and Sanger sequencing approaches, we have identified mutations in two different ARS genes. In the first family, homozygosity mapping in combination with subsequent mutation screening revealed a homozygous missense mutation in the  () gene (MIM# 607529), which co segregates with moderate ID,  <span style="color:red" title="HPO: HP:0000252
CONF: 0.9436223
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >microcephaly</span> ,  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >ataxia</span> ,  <span style="color:red" title="HPO: HP:0002167
CONF: 0.96763647
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >speech impairment</span> , and  <span style="color:red" title="HPO: HP:0000718
CONF: 0.99652386
PAT: None
NEG: False
REL: False
SECTION: 1 INTRODUCTION" >aggressive behavior</span> . The variant affects an amino acid residue that is highly conserved across the animal kingdom and it leads to unstable protein expression in mammalian cells. Moreover, the missense change leads to reduced enzymatic activity .<br>In the second family, we found compound DNA alterations affecting the mitochondrial  () gene (MIM# 604733) by whole genome sequencing and could show that both co segregate with moderate ID,  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: True
REL: False
SECTION: 1 INTRODUCTION" >ataxia</span> ,  <span style="color:red" title="HPO: HP:0002167
CONF: 0.96763647
PAT: None
NEG: True
REL: False
SECTION: 1 INTRODUCTION" >speech impairment</span> ,  <span style="color:red" title="HPO: HP:0000252
CONF: 0.9436223
PAT: None
NEG: True
REL: False
SECTION: 1 INTRODUCTION" >microcephaly</span> , and  <span style="color:red" title="HPO: HP:0000718
CONF: 0.99652386
PAT: None
NEG: True
REL: False
SECTION: 1 INTRODUCTION" >aggressive behavior</span> . The frameshift mutation on the paternal allele (c.325delA, p. Ser109Alafs*159) very likely leads to the degradation of  mRNA by NMD, given its position relative to the terminal exon. The second change (c.37T&gt:G, p. Trp13Gly), on the  <span style="color:red" title="HPO: HP:0002686
CONF: 0.93650734
PAT: None
NEG: False
REL: True
SECTION: 2 MATERIALS AND METHODS" >maternal</span>  allele, affects a mitochondrial signal peptide (SP) leading to mislocalization of the mutant protein in the cells.<br>and  add to the list of ubiquitously expressed genes implicated in ARID.<br>Furthermore, our results underscore the importance of ARSs in neuronal function and  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: True
SECTION: 2 MATERIALS AND METHODS" >brain</span>  development.<br><h2>2 MATERIALS AND METHODS</h2> <br> <br> <h3>2.1 Subjects and sampling</h3> <br> The study was approved by the Ethics Committee of the University of Social Welfare and Rehabilitation Sciences, Tehran, Iran. The patients and their families were recruited by the Genetics Research Center at the University of Social Welfare and Rehabilitation Sciences, Tehran, Iran, where genetic counselors obtained informed consent of the parents from both families in writing. The probands were examined by experienced clinical geneticists who assessed their physical and mental status. Blood samples for DNA preparation were collected and genomic DNA was extracted using a standard method.<br> <br> To exclude chromosomal abnormalities and metabolic disorders, respectively, we performed karyotype analysis by G banding and tandem mass spectrometry, using filter dried blood from at least one patient of each nuclear family. Fragile X syndrome was also excluded.<br><h3>2.2 Linkage analysis and autozygosity mapping</h3> <br> In the case of family 1, linkage analysis and autozygosity mapping were carried out. Genotyping (SNP analysis) was performed using 50k Genome Wide Human SNP Array Version 2 (Affymetrix, Santa Clara, CA) (Kennedy et al., 2003) based on previously published protocols (Matsuzaki et al., 2004) for all available individuals from family 1. We used Alohomora 11 software v0.32 (Ruschendorf &amp: Nurnberg, 2005) for SNP array quality controls, as described previously (Garshasbi et al., 2006). The program Merlin was applied for parametric multipoint linkage analysis consistent with an AR mode of inheritance, a disease allele frequency of 10 and complete penetrance (Abecasis, Cherny, Cookson, &amp: Cardon, 2002).<br><h3>2.3 Mutation screening and evaluation</h3> <br> Mutation screening for family 1 was performed by PCR analysis and subsequent Sanger sequencing of all exons and exon-intron boundaries of all protein coding genes within the linkage interval, which was flanked by the heterozygous SNPs rs10494061 and rs1938250 and contained 133 RefSeq genes (human reference genome, assembly GCRh37/hg19), using the DNA of the index patient (V:1). Primers were designed by using Primer3 software (URL: http://frodo.wi.mit.edu.proxy.library.uu.nl/cgi-bin/primer3/primer3_www.cgi) (Rozen &amp: Skaletsky, 2000). In case of the only plausible candidate gene, , PCR was performed in independent PCR reactions (primers are listed in Supp. Table S1). In addition, siblings and parents were also sequenced, when available, to confirm appropriate co segregation of the allele within the families. Furthermore, ethnically matched unrelated healthy individuals as well as unrelated healthy German individuals were also directly screened for the mutations. Moreover, we used the publically available sequencing databases as well as in house databases containing more than 400 exomes to further extend the control cohort. The DNA mutation numbering system used is based on the cDNA sequence where +1 is the A of the ATG translation initiation codon in the reference sequence.<br>In silico prediction of pathogenic effect of the variants identified in this study was performed using Polyphen2 (Adzhubei et al., 2010), SIFT (Ng &amp: Henikoff, 2001), Mutation Taster (Schwarz, Rodelsperger, Schuelke, &amp: Seelow, 2010), and Logit score developed by Hu et al. (2014).<br>We employed the PyMOL software (The PyMOL Molecular Graphics System, Version 1.5.0.1 Schrodinger, LLC) to model the protein structures of the mutant SARS based on the wild type (WT) crystal structures (3VBB) available in the Protein Data Bank PDB (Xu et al., 2012) and performed an in silico analysis to estimate the effect of the mutations on protein stability using Concoord/PBSA (Benedix, Becker, de Groot, Caflisch, &amp: Bockmann, 2009: Potapov, Cohen, &amp: Schreiber, 2009), DUET (URL: http://bleoberis.bioc.cam.ac.uk/duet), SCide (URL: http://www.enzim.hu/scide/), PoPMuSiC (URL: http://www.hsls.pitt.edu/obrc/index.php?page = URL1258139151), MUpro (http://mupro.proteomics.ics.uci.edu.proxy.library.uu.nl/), and I Mutant 2.0 (URL: http://gpcr.biocomp.unibo.it/cgi/predictors/I-Mutant2.0/I-Mutant2.0.cgi).<br>Family 2 was part of a large study of the molecular basis of ARID and genotyping results of six DNA samples (IV:2, IV:3, V:2, V:7, V:8, V:9) from this family were used for homozygosity mapping and multipoint parametric and non parametric linkage analysis. We identified three homozygous genomic intervals with LOD scores above 2 (Family M153 (Najmabadi et al., 2011)). Haplotype construction showed homozygous SNP haplotypes for all affected members within these three intervals. Subsequently, the three homozygous regions were enriched and next generation sequencing was performed but upon filtering no homozygous apparently disease causing mutation could be identified. More recently, this negative result was confirmed using the medical re sequencing analysis pipeline Medical Re sequencing Analysis Pipeline (MERAP) (Hu et al., 2014) (URL: https://sourceforge.net/projects/merap). This prompted us to perform whole genome sequencing to search for causative mutations even outside the homozygous regions. Whole genome sequencing was performed in one affected (V:7) by Complete Genomics (Mountain View, CA) as previously described (Drmanac et al., 2010). The mappable reads sum up to 160G, and the sequence fraction with &gt:10x coverage was 0.976 and with &gt:20x coverage was 0.936. Sequence reads were mapped to the reference genome (human reference genome, assembly GCRh37/hg19). Variants were ranked and prioritized as potential candidates as previously described (Najmabadi et al., 2011), using an improved version of MERAP (Hu et al., 2014). The OMIM catalogue (URL: https://www.omim.org/) and the Human Gene Mutation Database (HGMD, URL: http://www.hgmd.org/) were used as a filter to identify all previously described pathogenic changes. Furthermore data were filtered against multiple publically available databases including dbSNP138, sequencing data from the Exome Variant Server (NHLBI Exome Sequencing Project (ESP), Seattle, WA, URL: http://evs.gs.washington.edu/EVS/), 200 Danish individuals (Li et al., 2010), 185 genomes of healthy individuals made available by the 1,000 Genome Project (URL: http://www.1000genomes.org/) and Exome Aggregation Consortium (ExAC), Cambridge, MA (URL: http://exac.broadinstitute.org) [January 13, 2015]. We retained variants that were present with a minor allele frequency below 0.01. In addition, variants were compared with in house databases containing more than 400 exomes. The mutations within the candidate gene  were verified by Sanger sequencing (primers are listed in Supp. Table S1). Siblings and parents were also sequenced, when available, to confirm co segregation of the alleles within the family. Furthermore, ethnically matched unrelated healthy individuals as well as unrelated healthy German individuals were also directly screened for the mutations. DNA mutation numbering system used is based on cDNA sequence where +1 is the A of the ATG translation initiation codon in the reference sequence.<br>In silico analysis of the putative pathogenic missense change was performed using Project HOPE (URL: http://www.cmbi.ru.nl/hope/). Prediction of the pathogenic effect of the variant found in this study was performed using various softwares:<br><h3>2.4 Sanger sequencing</h3> <br> Sequencing reactions were prepared using the ABI PRISM Big Dye Terminator Cycle Sequencing V3.1 Ready Reaction Kit (Life Technologies, Thermo Fisher, Waltham, MA) and sequenced on a 3500 Genetic Analyzer for Resequencing &amp: Fragment Analysis (Applied Biosystems, Thermo Fisher, Waltham, MA). Subsequently, data were analyzed using CodonCode Aligner Software (CodonCode Corporation, Centerville, MA).<br><h3>2.5 RNA sequencing</h3> <br> Expression levels of mRNA corresponding to  (NM_006513.3) and  (NM_015836.3) were obtained by RNA sequencing of commercially available RNA samples from different  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: True
SECTION: 2.5 RNA sequencing" >brain</span>  tissues. RNA sequencing was carried out on a SOLiD5500XL Sequencing platform (Life Technologies, Thermo Fisher Scientific, Waltham, MA, USA) as previously described (Konze et al., 2014).<br><h3>2.6 Cloning of  and  cDNAs<br></h3> <br> A full length  cDNA clone (IRAUp969H0860D) was purchased from ImaGenes GmbH (Berlin, Germany). The cDNA was isolated from a  <span style="color:red" title="HPO: HP:0003006
CONF: 0.9881484
PAT: None
NEG: False
REL: True
SECTION: 2.5 RNA sequencing" >neuroblastoma</span>  cell line and contained a change from cytosine to thymine at c.1530, resulting in a missense mutation from arginine to cysteine at position 435 in the SARS protein. To correct the c.1530C&gt:T change, mutagenesis was performed with the QuickChange(tm) II XL Site Directed Mutagenesis Kit (Stratagene, La Jolla, CA).<br>The  complete cDNA was amplified using primers containing HI and I restriction sites and then cloned in the mammalian expression vector pEYFP N1, and in the vector for protein expression in prokaryotic organism, pGEX6P3. Primers (listed in Supp. Table S2) were designed using the QuikChange Primer Design Program (URL: http://www.stratagene.com/qcprimerdesign). To obtain  c.514G&gt:A (p. Asp172Asn) and c.1285A&gt:G (p. Thr429Ala), site directed mutagenesis was performed as described using sense and antisense primers carrying the respective nucleotide changes (see Supp. Table S2)<br>complete cDNA was obtained by PCR using primers WARS2 cDNA FW and WARS2 cDNA RV using total RNA from human fetal  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: True
SECTION: 2.5 RNA sequencing" >brain</span>  (BioChain, San Francisco, CA, USA). The PCR product was also used as template for further amplification with primers WARS2 EGFP FW and WARS2 EGFP RVN containing I and dIII restriction sites (see Supp. Table S2). After restriction digestion the fragment was inserted into the mammalian expression vector pEYFP N1. To obtain the  c.37T&gt:G (p. Trp13Gly) change, mutagenesis was performed with the QuickChange(tm) II XL Site Directed Mutagenesis Kit (Stratagene) (see Supp. Table S2 for primer sequences).<br><h3>2.7 Cell culture and treatment</h3> <br> COS7 and HEK293T cells were maintained in DMEM medium containing 10% FBS, and supplemented with l glutamine and penicillin/streptomycin. All transient transfection experiments were performed with Lipofectamine 2000 (Invitrogen, Thermo Fisher, Waltham, MA, USA) according to the manufacturer's recommendations. 5 x 10 COS7 cells were seeded in six well plates and transfected with 0.5 mg plasmid DNA per well and 2 ml lipofectamine, whereas 1 x 10 cells were seeded in 75 cm2 flask, transfected with 5 mg DNA per flask and 15 ml transfection reagent. After 24 hr, cells were fixed for immunofluorescence microscopy analysis or harvested. To stain mitochondria in live cells, cells were incubated with 250 mM MitoTracker(r) Deep Red FM (M22426: Invitrogen, Thermo Fisher Scientific, Waltham, MA, USA) for 20 min before harvesting them.<br><h3>2.8 Generation of whole cell lysates</h3> <br> WT and mutant constructs were transfected into HEK293T and/or COS7 cells. After 24 hr, cells were harvested by centrifugation at 110 x  for 5 min using a Rotanta 46K centrifuge (Hettich, Tuttlingen, Germany) and washed once in 1 x PBS. The pellet was resuspended into lysis buffer (15 mM Tris-HCl, pH 7.5, 48% urea, 8.7% glycerol, 1% SDS, 0.004% bromophenol blue, and 143 mM b mercaptoethanol) and then sonicated 10 sec/two cycles/max power with Sonoplus HD 2070 (Bandelin, Berlin, Germany).<br><h3>2.9 Cell fractionation</h3> <br> EYFP SARS WT and mutant expression vectors were transfected into HEK293T cells. After 24 hr, cells were harvested by centrifugation at 1,000 rpm for 5 min using a Rotanta 46K centrifuge (Hettich) and washed once in 1 x PBS. The pellet was resuspended in 400 ml Cell Fractionation Lysis Buffer (20 mM Hepes, pH 7.4, 50 mM NaCl, 5 mM MgCl2 protease inhibitors and 80 mg/ml digitonin) and incubated at RT for 10 min, then centrifuged at 956 x g for 10 min (Centrifuge 5417R: Eppendorf, Hauppauge, NY). The supernatant, which contained the cytosolic fraction, was removed and the pellet was washed twice with Cell Fractionation Lysis Buffer. The pellet, which contained the membrane and nuclear fraction, was resuspended in 150 ml of lysis buffer (15 mM Tris-HCl, pH 7.5, 48% urea, 8.7% glycerol, 1% SDS, 0.004% bromophenol blue, and 143 mM b mercaptoethanol) followed by sonication.<br><h3>2.10 Mitochondria isolation</h3> <br> Mitochondria were isolated using the Mitochondria Isolation Kit for Cultured Cells (Thermo Fisher Scientific, Waltham, MA, USA) following the manufacturer's instructions (option B). Modifications were done as follow: Step 3, cells were lysed by sonication 5 sec/3 x 2 cycles/max power. Step 8 and 11, pellets contained nuclei/membranes and mitochondria, respectively, were dissolved in 200 ml of lysis buffer (see above) and passed through QiaShredder (Qiagen, Venlo, Netherlands) to reduce viscosity.<br> <br> Protein concentration was measured by NanoDrop (ND 1000: Thermo Fisher Scientific, Waltham, MA, USA).<br><h3>2.11 Western blot</h3> <br> Protein extracts (5-25 mg) were loaded on 10% to 15% SDS Gels. The gels were blotted for 1 hr at 20 V. Membranes were blocked at RT either in 5% BSA or 5% milk in PBST for 1 hr. Incubation with 1st antibody in 5% milk in PBST was done overnight at 4degC. Membranes were incubated with secondary antibodies, diluted in PBST, for 1 hr at room temperature. To detect immunolabeled proteins, we used Western Lightning(r)ECL, Enhanced Chemiluminescence Substrate (PerkinElmer Inc., Waltham, MA, USA).<br><h3>2.12 Immunocytochemistry</h3> <br> Cells were fixed for 10 min with 4% PFA in PBS, permeabilized in 0.2% Triton X in PBS for 5 min, and incubated with 4% BSA in PBS for 1 hr at room temperature. They were then incubated overnight with the primary antibodies in the same solution at 4degC, washed three times with PBS and subsequently incubated with secondary antibodies in blocking solution (1 hr), and again washed three times with PBS, dipped in water and mounted with Fluoromount G (Southern Biotech, Birmingham, AL) and 0.5 mg DAPI/ml. Images were acquired using a confocal laser scanning microscope (Laser Scanning Microscope LSM700, Carl Zeiss, Oberkochen, Germany) with a 63x objective and were analyzed with the Carl Zeiss ZEN 2012 software.<br><h3>2.13 Antibodies</h3> <br> The following antibodies were used: anti SARS, clone 1H4: monoclonal, raised in mouse Abnova: anti GFP (Roche, Rotkreuz, Switzerland, cat. no.11814460001, monoclonal, mouse, IF: 1:1,000, WB: 1:5,000), anti Lamin A+C (Abcam, Cambridge, UK: cat. no. ab40567, monoclonal, mouse, WB: 1:250), anti Tubulin (abcam cat. no. ab6160, monoclonal, rat WB: 1:15,000), anti PBH1 (Cell Signaling Technology, Danvers, MA, USA: cat. no. #2426, polyclonal, rabbit, WB: 1:2,000), anti HSP60 (Cell Signaling Technology, Danvers, MA, USA: cat. no. #12165, monoclonal, rabbit, IF: 1:2,000, WB: 1:2,500), anti VDAC (Cell Signaling Technology: cat. no. #4661), anti COXIV (Cell Signaling Technology, Danvers, MA, USA: cat. no. #4850, monoclonal, rabbit, WB: 1:1,000), anti SDHA (Cell Signaling Technology, Danvers, MA, USA: cat. no. #11998, monoclonal, rabbit, IF and WB: 1:2,000). All HRP conjugated antibodies (Santa Cruz, Dallas, TX, USA) were used in 5% milk/PBST. Fluorescent labeled antibodies anti mouse AlexaFluor488 and anti rabbit AlexaFluor594 were purchased from Molecular Probes (Thermo Fisher Scientific, Waltham, MA, USA: cat. no. A21200 and A11037, respectively) and used with a dilution of 1:1,000.<br><h3>2.14 Expression and purification of GST fusion proteins</h3> <br> The pGEX6P3 vector was used for high level inducible intracellular expression of SARS WT and mutant proteins in  BL21. Expression of GST fusion protein was induced by addition of 1 mm IPTG for 4 hr at 37degC. GST Protein purification was done using the GST Bulk Kit (GE Healthcare, Little Chalfont, UK). In summary, cell pellets were dissolved in ice cold 1 x PBS containing an appropriate amount of complete protease inhibitor tablets, were sonicated using an ultrasonic cell disruptor (Bandelin) at maximum power for 10 sec and two cycles. Afterward, sonicates were centrifuged to remove insoluble material (500, 4degC, 20 min). Glutathione Sepharose 4B was prepared according to the manufactures instructions. Two microliters of 50% Glutathione Sepharose 4B slurry was added to each 100 ml of lysate sample. The mixture was incubated at 4degC on an overhead shaker overnight. GST fusion proteins were eluted by addition of 0.5 ml elution buffer per 1 ml of Glutathione Sepharose 4B. Elution was repeated for five to six times and protein concentration was determined by silver staining.<br>Silver staining was performed using ProteoSilver(tm) Plus Silver Stain Kit (Sigma-Aldrich, Munich, Germany) following manufacturer's instructions. For determining protein concentration in gel bands, gels were scanned and analyzed using the ImageQuant software (Molecular Dynamics, GE Healthcare, Little Chalfont, UK).<br> <br> Buffer exchange was performed with PD 10 Desalting Columns (GE Healthcare, Chicago, IL, USA). The protocol by gravity flow was applied according to the manufacturer's instructions.<br><h3>2.15 Pyrophosphate release assay</h3> <br> Pyrophosphate release of purified recombinant GST SARS WT and mutant (p. Asp172Asn and p. Thr429Ala) proteins and GST protein was assayed at 37degC for 90 min. Per reaction, 12.5 ng proteins were used in a total volume of 300 ml. Negative controls contained buffer eluted from PD 10 desalting columns instead of protein.<br> <br> Pyrophosphate release was determined at three different time points (0, 60, and 90 min). Reactions were stopped approximately simultaneously by adding twofold reaction volume of BIOMOL GREEN Reagent(tm) (Enzo Life Sciences, Lorrach, Germany) to each sample. Samples were then incubated at RT for 20 min to allow for the development of the green color. Subsequently, 300 ml of this mix per sample was pipetted in triplicate into a well of a 96 well EIA/RIA Plate (Corning, Corning, NY) and OD620 was measured in a microplate reader (BMG Labtech, Ortenberg, Germany). Standards were prepared in pyrophosphate release reaction buffer that had been pre incubated for 90 min at 37degC. After incubation, serial dilutions were prepared to obtain concentrations of 2, 1, 0.5, 0.25, 0.125, 0.63, and 0.31 nmol phosphate, respectively. Standard sample reactions were terminated at the same time as assay sample reactions using twofold reaction volume of BIOMOL GREEN Reagent(tm) (Enzo Life Sciences, Lorrach, Germany). Standard samples were prepared in triplicate for each  <span style="color:red" title="HPO: HP:0100529
CONF: 0.8132653
PAT: None
NEG: False
REL: True
SECTION: 3 RESULTS" >phosphate concentration</span> .<br><h2>3 RESULTS</h2> <br> <br> <h3>3.1 Clinical report of family 1</h3> <br> Family 1 is an Iranian family with Arabian ethnicity. The distantly related parents had nine children, four of them presented with ID: the pedigree is shown in Figure 1A. All affected children were born at full term after uneventful pregnancies, followed by a normal neonatal period. Testing of  <span style="color:red" title="HPO: HP:0100543
CONF: 0.95951575
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >cognitive</span>  function using the Wechsler Adult Intelligence Scale (WAIS IV)  <span style="color:red" title="HPO: HP:0001249
CONF: 0.8140416
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >showed an intelligence</span>  quotient (IQ) 40-45 in the range of moderate ID,  <span style="color:red" title="HPO: HP:0002167
CONF: 0.96763647
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >speech impairment</span>  (slow and  <span style="color:red" title="HPO: HP:0001350
CONF: 0.9699675
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >slurred speech</span> ) and  <span style="color:red" title="HPO: HP:0000718
CONF: 0.99652386
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >aggressive behavior</span> . In addition, the patients presented with  <span style="color:red" title="HPO: HP:0000252
CONF: 0.9436223
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >microcephaly</span> ,  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >ataxia</span>  (unbalanced  <span style="color:red" title="HPO: HP:0001288
CONF: 0.97701836
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >gait</span>  starting at the age of 7-10 years), and  <span style="color:red" title="HPO: HP:0001324
CONF: 0.9807812
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >muscular weakness</span>  (since the age of 6-10). All patients developed  <span style="color:red" title="HPO: HP:0002069
CONF: 0.98978853
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >tonic-clonic</span>   <span style="color:red" title="HPO: HP:0001250
CONF: 0.98365414
PAT: None
NEG: False
REL: True
SECTION: 3.1 Clinical report of family 1" >seizures</span>  during infancy, which are well controlled by sodium valproate. DNA was extracted from peripheral blood of the patients (III:4, III:9, III:10, III:11), three healthy siblings and their parents, using standard procedures. Clinical descriptions of the patients are summarized in Table 1.<br><h3>3.2 Clinical report of family 2</h3> <br> Family 2 is an Iranian family from the Semnan province of Iran. The consanguineous healthy parents had nine children, three of them with ID (Fig. 1B). One affected male (V:6) had died from an infectious disease. Two affected siblings (V:7 and V:8) presented with moderate ID (IQ = 41-46),  <span style="color:red" title="HPO: HP:0002305
CONF: 0.9866977
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >athetosis</span> ,  <span style="color:red" title="HPO: HP:0000718
CONF: 0.99652386
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >aggressive behavior</span> , and  <span style="color:red" title="HPO: HP:0002167
CONF: 0.96763647
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >speech impairment</span>  (slow and  <span style="color:red" title="HPO: HP:0001350
CONF: 0.9699675
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >slurred speech</span> ). Moreover  <span style="color:red" title="HPO: HP:0001324
CONF: 0.9807812
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >muscular weakness</span>  and  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >ataxia</span>  (unbalanced  <span style="color:red" title="HPO: HP:0001288
CONF: 0.97701836
PAT: v:7
NEG: False
REL: True
SECTION: 3.2 Clinical report of family 2" >gait</span>  starting at the age of 7-10 years) were observed (see Table 1 for full description of clinical features). DNA was extracted from peripheral blood of the patients, two healthy siblings and their parents, using standard procedures.<br><h3>3.3 Linkage analysis and homozygosity mapping in family 1</h3> <br> Parametric linkage analysis in family 1 revealed a single interval of homozygosity on chromosome 1p13.3-p11.2 near the centromere with a significant LOD score above three (Supp. Fig. S1). The linkage interval had a size of 12.9 Mb, was flanked by the heterozygous SNPs rs10494061 and rs1938250, and contained 133 protein coding RefSeq genes (human reference genome, assembly GCRh37/hg19). For this study, coding regions including intron-exon boundaries of these genes were amplified by PCR and subsequently sequenced.<br><h3>3.4 Mutation analysis in family 1</h3> <br> In the exonic regions of all 133 genes included in the interval, only two homozygous missense variants co segregating with the disease in the family 1 were detected in the linkage interval on chromosome 1 (Supp. Fig. S2). In the  gene a homozygous G to A substitution (chr1: 109773566G&gt:A: NM_006513.3: c.514G&gt:A) (Fig. 1C) (coordinates used correspond to human reference genome, assembly GCRh37/hg19) was found in exon five and results in a change from aspartic acid to asparagine at amino acid position 172 (NP_006504.4: p. Asp172Asn). In addition, a variant (chr1: 120166494G&gt:T: NM_001080470.1: c.472C&gt:A) in the third exon of the zinc  <span style="color:red" title="HPO: HP:0001167
CONF: 0.8023411
PAT: None
NEG: False
REL: True
SECTION: 3.4 Mutation analysis in family 1" >finger</span>  protein 697 () gene was found, which results in a substitution of proline with threonine at amino acid position 158 (NP_001073939: p. Pro158Thr), but was predicted to be benign by several different pathogenicity evaluators (Polyphen2, SIFT, Mutation Taster, and Logit score). Only the amino acid substitution in SARS was predicted to be damaging and thus identified as the most plausible pathogenic change in family 1.  c.514G&gt:A was not present in dbSNP 138, 200 Danish individuals (Li et al., 2010) 185 genomes of healthy individuals made available by the 1000 Genome Project (1000 Genomes Project Consortium et al., 2010), the NHLBI Exome Variant Server (ESP6500), or in the ExAc database (Version 3.0). It was also absent from our in house databases containing more than 400 exomes, from more than 1,000 chromosomes from ethnically matched controls and from 432 chromosomes of specifically tested German controls. The variant was submitted to the public Leiden Open Variation Database 3.0 (http://www.lovd.nl/).<br>The mutation affects a highly conserved amino acid (Fig. 1C) and, in line with this, the phyloP score (Pollard, Hubisz, Rosenbloom, &amp: Siepel, 2010) was found to be 6.457.<br>Expression of  was determined by RNAseq in different human  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: True
SECTION: 3.4 Mutation analysis in family 1" >brain</span>  regions relevant for learning and memory (Supp. Fig. S3).<br>SARS encodes the cytosolic seryl tRNA synthetase, a protein that contains three domains: an N terminal tRNA binding arm, an aminoacylation domain and a C terminal UNE S domain containing the nuclear localization signal (NLS) ADDIN EN. CITE (Xu et al., 2012). The amino acid substitution p. Asp172Asn found in family 1 is located in the aminoacylation core domain (Fig. 1C), in close proximity to the active site (see also Supp. Fig. S4).<br><h3>3.5 The amino acid substitution p. Asp172Asn destabilizes SARS</h3> <br> Mutations of single amino acid mutations can significantly change the stability of a protein structure. The p. Asp172 was predicted in silico to be a stabilization center element by the SCide software (Dosztanyi, Magyar, Tusnady, &amp: Simon, 2003) using PDB structure 3VBB and chain F. To understand whether the stability of SARS protein was influenced by the loss of the negative charge at residue 172, protein stability was assessed by MUpro, I Mutant (DD = -2.06) and PoPMuSiC (DD = 0.62 kcal/mol). All three tools predicted a moderate decrease in protein stability upon mutation. Even more important, in silico energy calculations with the Concoord/PBSA method (Benedix et al., 2009), which calculates stability changes upon mutation confirmed a decrease in stability (DD = -1.06).<br>To gain insights into the structural consequences of the p. Asp172Asn mutation, structure based  modeling of the p. Asp172Asn amino acid substitution was performed using the crystal structure 3VBB of human SARS (Xu et al., 2012). p. Asp172 is located in the aminoacylation domain, in close proximity to p. His170 (Supp. Fig. S3A). Xu et al. (2012) have also shown that residues p. His170 and p. Phe316 form hydrophobic interactions to stabilize the b 10-b 11 hairpin. This hairpin belongs to the seven stranded antiparallel b sheet (b1-b9-b10-b11-b13-b8-b7), which is the central core of the aminoacylation domain (Xu et al., 2012).<br>Aspartic acid and asparagine are geometrically similar and protein modeling showed that p. Asp172 faces away from the active site (Supp. Fig. S4A), therefore p. Asp172Asn has probably no structure based influence on the catalytic core domain. However, p. Asp172 could affect aminoacylation activity in an indirect manner as it provides a negative charge. This negative charge is lost in SARS p. Asp172Asn. Furthermore, the mutation site p. Asp172 is located directly adjacent to the N terminus of the UNE S domain, which contains the NLS (Supp. Fig. S4B and C). Negative charges within the UNE S domain affect the stability of the positively charged NLS (Xu et al., 2012). Therefore, the substitution of the negatively charged p. Asp172 by the positively chargedp. Asn172 could influence the subcellular localization of the SARS protein.<br><h3>3.6 Expression analysis of SARS p. Asp172Asn in mammalian cells</h3> <br> To test the hypothesis that p. Asp172Asn might influence the localization pattern of SARS, we cloned WT and mutant SARS in frame with C  terminal EYFP tag. In parallel, subcellular localization of endogenous SARS was analyzed using a monoclonal SARS antibody. Confocal microscopy revealed a similar cytoplasmic localization of endogenous and ectopic SARS proteins in all cell lines tested (data not shown) which is in agreement with previous studies (Xu et al., 2012).<br>Subsequently, we transfected HEK293T cells with EYFP SARS WT and p. Asp172Asn expression vectors for 24 hr and transfection efficiency was determined in parallel by counting transfected cells by fluorescence microscopy. Transfections rates were 40.8 +- 3.2% for SARS WT and 39.8 +- 1.1% for SARS p. Asp172Asn, respectively. Data to evaluate transfection efficiency were derived from three independent transfections.<br> <br> After transfection, cells were harvested and total protein lysate as well as cytosolic and nuclear protein fractions were extracted. Expression of ectopic WT and mutant SARS proteins were analyzed by Western blotting using monoclonal anti GFP antibody (Fig. 2A upper panel). The same membranes were then hybridized with anti tubulin (Fig. 2A middle panel) and anti lamin antibodies (Fig. 2A bottom panel). Tubulin and lamin were used as control for cell fractionation and normalization.<br>Quantification of six independent experiments revealed that expression of p. Asp172Asn was significantly reduced as compared with the WT in whole cell extracts as well as in the cytosolic fraction (&lt: 0.005) (Fig. 2B). In the nuclear fraction, signals were very weak and differences did not reach a level of statistical significance.<br><h3>3.7 Impaired serine activation of the SARS p. Asp172Asn enzyme</h3> <br>  modeling of SARS revealed that p. Asp172 maps close to the active site of the protein (Supp. Fig. S3A) in the tRNA synthetase class II core domain (Fig. 1C). Although it is facing away from the active site, p. Asp172 might influence SARS aminoacylation activity because of its negative charge, which is lost upon mutation (p. Asp172Asn). Aminoacylation is performed by ARSs in a two step reaction: first, an ARS binds an amino acid and ATP to form an aminoacyladenylate (AA AMP). At the same time, pyrophosphate (P2O7, PPi) is released. In the second step, the amino acid is transferred onto the 3' terminal adenosine of the tRNA that is to be charged and subsequently the charged tRNA and the AMP molecule are released from the enzyme (Schimmel &amp: Soll, 1979). To understand whether mutant SARS is still able to perform the first step of aminoacylation, we established a pyrophosphate release assay. As prerequisite  WT and mutant (p. Asp172Asn and p. Thr429Ala) cDNAs were cloned into pGEX6P3. SARS protein with a p. Thr429Ala mutation is unable to bind serine as this alteration affects a serine binding site (Fig. 1C) and lacks enzymatic activity (Fukui, Hanaoka, &amp: Kawahara, 2009). This protein was used as a negative control during the aminoacylation test.<br>The purified GST protein was used as an additional negative control. For each protein, activity measurements were performed at three different time points (0, 60, and 90 min) (Fig. 2C). The pyrophosphate release of the WT SARS reached a maximum at 60 min, and after 90 min of incubation the enzyme still released approximately two thirds of the maximal amount of pyrophosphate as a by product of serine activation. Firstly, this reduced rate of pyrophosphate production can be explained by entering of AMP into the back reaction of amino acid activation. It was shown by Rapaport, Remy, Kleinkauf, Vater, and Zamecnik (1987) that this mechanism is an inherent part of the catalytic activities of ARSs and could prevent the buildup of enzyme bound or free aminoacyl adenylates. Secondly, substrate inhibition might cause the decrease in aminoacylation activity because seryl adenylate canremain bound to the SARS enzyme. During aminoacylation of  alanyl tRNA synthetase (AARS), the alanyl adenylate product remains bound tightly to the enzyme and is only turned over when tRNA is present in the reaction. In addition, pyrophosphate generated in the adenylation step substantially inhibits the initial rate of aminoacylation of tRNA in a dose dependent manner. The degree to which pyrophosphate inhibits a given reaction depends, inter alia, on the extent to which the reaction has progressed. However, the mechanism of this inhibition still needs to be established (Wolfson &amp: Uhlenbeck, 2002). Remarkably, a highly significant loss of serine activation was observed for the SARS p. Asp172Asn mutant. At 60 min, GST SARS p. Asp172Asn released only 28% pyrophosphate as compared with the GST SARS WT enzyme and even after 90 min of incubation GST SARS p. Asp172Asn still produced 51% less pyrophosphate than the WT enzyme at 60 min. Consistent with previous findings (Xu et al., 2012), the GST SARS p. Thr429Ala was inactive during the assay (Fig. 2C). As expected, the GST protein did not show any pyrophosphate release (data not shown).<br><h3>3.8 Compound heterozygous  mutation in family 2<br></h3> <br> As a result of whole genome sequencing, we found a compound heterozygous genotype of the  gene in the affected individuals of family 2 (Fig. 1B and D).<br>WARS2 is a nucleus encoded tryptophanyl tRNA synthetase, which is imported into mitochondria. The protein contains an N terminal mitochondrial targeting signal, and a C terminal domain with tryptophan tRNA ligase activity (Fig. 1D). Furthermore, it is probably able to load tRNA(Sec) with serine, which would result in a misacylated tRNA, L seryl tRNA(Sec) that can be further converted into selenocysteinyl tRNA(Sec). Expression of  was evaluated by RNAseq in different human  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: True
SECTION: 3.7 Impaired serine activation of the SARS p. Asp172Asn enzyme" >brain</span>  regions relevant for learning and memory (Supp. Fig. S5).<br>The WARS2 genotype we observed in the affected individuals comprises a frameshift mutation on the paternal allele and a missense change on the  <span style="color:red" title="HPO: HP:0002686
CONF: 0.93650734
PAT: None
NEG: False
REL: True
SECTION: 3.7 Impaired serine activation of the SARS p. Asp172Asn enzyme" >maternal</span>  allele. The mutations cosegregated with the disease in the family (Supp. Fig. S6 and Supp. Table S3). Both were submitted to the public Leiden Open Variation Database 3.0 (http://www.lovd.nl/). The frameshift mutation in exon 2 of the gene (chr1:119618996delA, NM_015836.3: c.325delA, NP_056651: p. Ser109Alafs*15) leads to a premature stop codon and, given its location relative to the terminal exon 6, is most likely to cause nonsense mediated decay (NMD) (Fig. 1D), resulting in a complete loss of a functional gene product from this allele. The change was not found in the various control panels used (see p.14)<br>The second change (chr1:119683231A&gt:C, NM_015836.3: c.37T&gt:G) (Fig. 1D) was also absent in controls (see p.14) and is located in exon 1 and is present in the ExAc database (Version 0.3) with an allelic frequency 0.003383. It affects an amino acid (NP_056651: p. Trp13Gly) located in an N terminal mitochondrial SP. In order to determine putatively adverse functional effects of this alteration on the remaining WARS2 allele, we first performed  3D modeling (Project HOPE) of the p. Trp13Gly variant, which indicated that the introduced glycine, being smaller and less hydrophobic than the WT Trp residue, indeed has an effect on WARS2 protein structure and function. The size difference between the WT and mutant residue renders it likely that the mutant Gly residue is not in the correct position to make the same hydrogen bonds as the original amino acid. In addition, the difference in hydrophobicity between Trp and Gly can affect the hydrogen bond formation as well.<br>This is important as p. Trp13Gly is located within the mitochondrial transit peptide. This peptide forms a special signal that is required to find the correct location of the protein in the cell. A mutation in this peptide might change or impair the recognition signal and therefore affect the correct localization of WARS2. To evaluate this assumption , we took advantage of several publicly available and widely used softwares including Predotar (score for WT: 0,69 and for p. Trp13Gly: 0,35), MITOPROT II (probability for WT: 0,7183and for p. Trp13Gly: 0,5184), SLP Local (SVM score for WT: 0,709, RI:1 and for p. Trp13Gly: 0,682, RI: 1), and TargetP 1.1 (score for WT: 0.887, RC: 2 and for p. Trp13Gly: 0,869, RC: 2), all of which predicted that the amino acid substitution would reduce the import into the mitochondria, and thus result in a partial mislocalization of WARS2 in the cells.<br><h3>3.9 Impaired mitochondrial localization of the WARS2 p. Trp13Gly protein</h3> <br> To experimentally test the hypothesis that the p. Trp13Gly change is functionally relevant, we first performed immunofluorescence microscopy experiments, which revealed that WARS2 EYFP and p. Trp13Gly WARS2 EYFP were present in the cytoplasm of COS7 cells, in dot like structures. To investigate whether these punctuated structures represent mitochondria, we treated live cells with MitoTracker, a red fluorescent dye that stains mitochondria. Furthermore, its accumulation is dependent upon mitochondrial membrane potential (Chazotte, 2011). Upon treatment, most WARS2 WT positive granules overlapped with the mitochondria staining, thus confirming previous findings (Jorgensen, Sogaard, Rossing, Martensen, &amp: Justesen, 2000), whereas a partial reduction in the overlapping signal was observed for the p. Trp13Gly mutant (Fig. 3A). This result was further substantiated by the results of a co staining experiment with the mitochondrial marker HSP60 (Fig. 3B).<br>To confirm the localization of WARS2 to the mitochondria and to further evaluate if p. Trp13Gly has a negative impact in this context, we performed subcellular fractionation to isolate mitochondria from HEK293T cells transiently expressing WARS2 EYFP and p. Trp13Gly WARS2 EYFP. Protein lysates from mitochondrial and cytosolic fractions were then submitted to SDS gel and Western blot analysis. This revealed that both WT and mutant WARS2 were enriched in the mitochondrial fraction. A specific antibody against PHB1, a mitochondrial marker, was used to confirm the quality of the subcellular fractionation (Supp. Fig. S7 and Fig. 4). To determine if the p. Trp13Gly change, which affects the mitochondrial SP in WARS2, could impair the protein translocation to the mitochondria, we quantified the amount of WT and mutant protein in the isolated mitochondrial fraction, using a panel of mitochondria specific proteins for normalization. This analysis revealed that the amount of p. Trp13Gly WARS2 in the mitochondria was significantly reduced as compared with the WT protein (Fig. 4A-E). Western blot analysis of total protein lysates extracted in parallel, showed no significant difference between total amount of WARS2 WT and p. Trp13Gly WARS2 (Fig. 4F and G). The transfection efficiency was determined by counting transfected cells by fluorescence microscopy. Transfections rates were 42.8 +- 2.2% for WARS2 WT and 40.8 +- 1.3% for WARS2 p. Trp13Gly, respectively. Data were derived from three independent transfections. Interestingly, by immunofluorescence analysis, using a specific antibody against SDHA, a catalytic subunit of succinate ubiquinone oxidoreductase, and part of the mitochondrial respiratory chain, we observed a marked decrease of SDHA signal (Supp. Fig. S8) in cells overexpressing WT WARS2 compared with cells overexpressing the mutant protein and control cells. Western blot analysis performed on the mitochondrial fraction confirmed this observation: the amount of SDHA was significantly reduced upon WT WARS2 overexpression as compared with mutant WARS2 (Fig. 4A and H).<br><h2>4 DISCUSSION</h2> <br> In this report, we describe putatively ID causing mutations in  and  in families with closely overlapping additional clinical features including moderate ID,  <span style="color:red" title="HPO: HP:0002167
CONF: 0.96763647
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >speech impairment</span> ,  <span style="color:red" title="HPO: HP:0001250
CONF: 0.98365414
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >seizures</span> ,  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >ataxia</span> , and  <span style="color:red" title="HPO: HP:0001324
CONF: 0.9807812
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >muscular weakness</span> . The occurrence of  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >ataxia</span>  is consistent with previous reports that implicate ARSs in the etiology of this particular feature (see e.g. (Lee et al., 2006)). Moreover,  <span style="color:red" title="HPO: HP:0001324
CONF: 0.9807812
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >muscular weakness</span>  was repeatedly observed in the context of tRNA related disorders in general or ARS related conditions in particular, for example,  <span style="color:red" title="HPO: HP:0002821
CONF: 0.97555494
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >Charcot</span> -Marie-Tooth (CMT) disease (MIM# 607731) and others (see e.g. (Abbott, Francklyn, &amp: Robey Bond, 2014: Bansagi et al., 2015: Irobi, De Jonghe, &amp: Timmerman, 2004: McLaughlin et al., 2012), which also supports our conclusion that the phenotype in our families is caused by the DNA alterations we report here.<br>The gene products of both genes are involved in tRNA aminoacylation and we could confirm that both are well expressed in tissues relevant for memory and learning in the mature  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >brain</span> . Together with AIMP1, which we have recently found to be implicated in ID without  <span style="color:red" title="HPO: HP:0002180
CONF: 0.9939998
PAT: None
NEG: True
REL: False
SECTION: 4 DISCUSSION" >neurodegeneration</span>  (Iqbal et al., 2016), there are now at least three genes with a role in tRNA aminoacylation that are apparently relevant for human  <span style="color:red" title="HPO: HP:0100543
CONF: 0.89597446
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >cognition</span> .<br>The  gene product catalyzes the attachment of serine to tRNA(Ser). Moreover it is likely able to charge tRNA(Sec) with serine, resulting in the misacylated tRNA L seryl tRNA(Sec), which can be further converted into selenocysteinyl tRNA(Sec). We found that the p. Asp172Asn alteration we identified leads to impaired serine activation of the SARS enzyme.<br>In the second family, whole genome sequencing revealed a compound heterozygous genotype in affected individuals, consisting of a deleterious frame shift mutation and a missense change (p. Trp13Gly) in the mitochondrial tryptophanyl tRNA synthetase encoding . We could show that the latter has a putative effect on structure and function of the gene product from the remaining allele and demonstrate impaired mitochondrial localization of the remaining WARS2 p. Trp13Gly protein. This might in itself have only a subliminal effect, which when present in a heterozygous state, may not have significant downstream consequences. However, when paired with a deleterious alteration on the second allele, such as the frameshift mutation we found, the subliminally impaired function of the missense allele might not be sufficient to compensate and thus result in a pathogenic reduction in WARS2 enzymatic activity. Similar observations were made by Sauter et al. (2015) who report that carriers of compound  mutations show the strongest effects concerning DARS2 solubility and activity, when missense mutations occur in combination with a splicing defect on the second allele.<br>Interestingly, ectopic expression of WT WARS2 in HEK293T cells induced a significant reduction in the amount of SDHA protein as compared with controls or to cells overexpressing the p. Trp13Gly variant of WARS2. Based on the canonical function of WARS2, this might suggest a role for this protein in the translation of SDHA, which would then be partially impaired in the mutant protein. This is noteworthy, since homozygous mutations in the  gene cause Leigh syndrome and Mitochondrial  <span style="color:red" title="HPO: HP:0008314
CONF: 0.9159447
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >respiratory chain complex II deficiency</span>  (MIM# 256000, MIM# 252011) in patients that present, among other features,  <span style="color:red" title="HPO: HP:0001251
CONF: 0.9794151
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >ataxia</span> ,  <span style="color:red" title="HPO: HP:0001324
CONF: 0.98386145
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >muscle weakness</span> ,  <span style="color:red" title="HPO: HP:0100543
CONF: 0.9882897
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >cognitive impairment</span> , and  <span style="color:red" title="HPO: HP:0001250
CONF: 0.98365414
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >seizures</span> .<br>We believe that WARS2 p. Trp13Gly is in fact a functional polymorphism (Albert, 2011) that is uncovered by a loss of the gene product from the second allele. In this context it should also be mentioned that position 13 in the WARS2 gene product can be affected also by one other alteration (p. Trp13Arg) but this is very rare (3/121412 alleles according to the ExAC database) and was not observed in a homozygous state. Moreover, albeit p. Trp13Gly was found in 403 of the 60,706 individuals included in the ExAC database (URL: http://exac.broadinstitute.org/) to date (August 2016) even this variant was found to be homozygous only once. The functional relevance of the p. Trp13Gly alteration as such might be primarily due to its impact on the amino acid composition of the localization signal sequence in the  gene product, which leads to a mislocalization of the otherwise functional protein. It is therefore conceivable that missense changes affecting the composition of this important molecular section in general might have a negative effect without changing the functional integrity of the affected protein per se (see e.g. also Hu et al., 2011). This is interesting because the disease causing potential of such alterations might in many cases not be identified by the presently available prediction algorithms, so that in NGS based mutation screening studies such polymorphisms are probably still being filtered out.<br>A recently generated targeted  rat model supports our hypothesis: While, like in uncharacterized knockout mice (URL: http://www.informatics.jax.org/), the  deficient allele is embryonically lethal in a homozygous state (), heterozygous animals () appear normal. Moreover, it has been shown that animals heterozygous () for the knockout allele and  p. Leu53Phe, a common variant in rat strains that is associated with a 40% reduction in Wars2 activity, revealed diminished cardiac angiogenesis and reduced coronary flow as compared with animals with compound heterozygosity for the WT allele and the p. Leu53Phe allele () (Wang et al., 2016). This suggests that  represents a compound hypomorph similar to the situation observed in our patients.<br>Functionally the mutations we found in SARS and WARS2 might lead to a reduced amount of charged tRNA available for translation at the ribosomes, which in turn could lead to a reduced translation rate, especially of those proteins that contain  <span style="color:red" title="HPO: HP:0500138
CONF: 0.81653136
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >high amounts of serine</span> , selenocysteine or tryptophan, respectively. Selenocysteine for example is an essential component of the 25 selenoproteins encoded in the human genome (Lu &amp: Holmgren, 2009). Selenoproteins are involved in protein folding, degradation of misfolded membrane proteins, and control of cellular calcium homeostasis (Andersen, 2004). Interestingly, deletion of selenoprotein P (Sepp1) in mice produces both neuronal and  <span style="color:red" title="HPO: HP:0040078
CONF: 0.9701006
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >axonal degeneration</span>  as well as potentially reversible neurite changes in the developing  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >brain</span>  (Caito et al., 2011). Neuron specific ablation of selenoprotein expression causes a neurodevelopmental and  <span style="color:red" title="HPO: HP:0002180
CONF: 0.9968527
PAT: None
NEG: True
REL: False
SECTION: 4 DISCUSSION" >neurodegenerative</span>  phenotype in mice,  <span style="color:red" title="HPO: HP:0002538
CONF: 0.8297024
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >affecting the cerebral cortex</span>  and the hippocampus, particularly the parvalbumin (PV) positive interneuron population (Wirth et al., 2010). Moreover, loss of PV neurons is associated with some  <span style="color:red" title="HPO: HP:0000708
CONF: 0.88457793
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >neuropsychiatric disorders</span> , for example,  <span style="color:red" title="HPO: HP:0000729
CONF: 0.98254406
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >autism spectrum disorders</span>  (Sgado et al., 2013).<br>Specifically with respect to SARS p. Asp172Asn, our overexpression experiments, which we carried out in HEK293T cells, because of their high transfection efficiency and because they are commonly used to assess an impairment of localization in the mitochondria, showed that in comparison to the ectopic WT protein the level of ectopic p. Asp172Asn was significantly reduced in whole cell extracts as well as in the cytosolic fraction. As transfection rates for both were similar this is most likely due to the mutation, suggesting reduced protein stability of mutant SARS, which is also in line with our 3D modeling results. Intriguingly, the observed amounts of WT and mutant SARS in the nuclear fraction were similar, indicating that while the mutation seems to affect the overall stability and function of the SARS gene product, the nuclear transport properties seem to be unaffected. Loss of SARS or severe impairment of SARS function might slow down overall translation rates during cellular processes that depend on fast and reliable protein translation (e.g., in axonal growth cones: reviewed by Jung &amp: Holt, 2011: Swanger &amp: Bassell, 2011) or synaptic processes involved in learning, memory processing, and memory storage as reviewed by Bekinschtein et al. (2010), Costa Mattioli, Sonenberg, and Richter (2009), and Gal Ben Ari et al. (2012). Similar disease mechanisms have been proposed for mutations within , which are associated with infantile hepatopathy (Casey et al., 2012), and mutations within  that cause recessive intermediate CMT disease type B (CMTRIB) (MIM# 613641) (McLaughlin et al., 2010) with a phenotype that does not only comprise  <span style="color:red" title="HPO: HP:0009830
CONF: 0.9717754
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >peripheral neuropathy</span>  but also self abusive behavior and  <span style="color:red" title="HPO: HP:0001263
CONF: 0.99637204
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >developmental delay</span> . Finally, it is also noteworthy that in the presence of a tryptophanyl tRNA deficiency,  <span style="color:red" title="HPO: HP:0002180
CONF: 0.9968527
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >neurodegenerative</span>  symptoms can be induced by tryptamine. Tryptamine induces axonopathy and mitochondriopathy mimicking  <span style="color:red" title="HPO: HP:0002180
CONF: 0.9968527
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >neurodegenerative</span>  diseases via tryptophanyl tRNA deficiency (Paley, Perry, &amp: Sokolova, 2013).<br>Regarding WARS2, it is interesting to note that mutations in human mitochondrial ARSs are associated with various  <span style="color:red" title="HPO: HP:0002180
CONF: 0.9968527
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >neurodegenerative</span>  diseases and that all other mitochondrial ARSs, were already found to be subject to pathology related mutations (Diodato, Ghezzi, &amp: Tiranti, 2014: Hallmann et al., 2014: Konovalova &amp: Tyynismaa, 2013: Schwartzentruber et al., 2014: Schwenzer, Zoll, Florentz, &amp: Sissler, 2014: Sofou et al., 2015: Suzuki, Nagao, &amp: Suzuki, 2011). Mitochondrial protein synthesis is central to the mitochondrial oxidative phosphorylation system (OXPHOS). Why clinical consequences of mutations in essential genes are restricted to the  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >brain</span>  is still unclear, however, it has been suggested before that, given the fundamental  <span style="color:red" title="HPO: HP:0003287
CONF: 0.82695293
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >function of mitochondrial</span>  ARSs, a loss of function is incompatible with life but that low enzyme activity may be sufficient to allow  <span style="color:red" title="HPO: HP:0000130
CONF: 0.8669952
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >uterine</span>  development yet still result in specific disorders during postnatal life (Konovalova &amp: Tyynismaa, 2013). The high demand of energy for the maintenance of  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >brain</span>  functions makes the  <span style="color:red" title="HPO: HP:0002011
CONF: 0.82331496
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >central nervous</span>  system particularly sensitive to disturbances in the (mitochondrial) OXPHOS and it is conceivable that  <span style="color:red" title="HPO: HP:0100543
CONF: 0.95951575
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >cognitive</span>  functions, being the last to develop during evolution, suffer first from imbalances in the energy supply system. When present at crucial neurodevelopmental stages such disturbances could result in permanent damage, while other tissues or functions might not be affected, the residual WARS2 activity being sufficient to cover the demand in these cases.<br>Furthermore, synapses might be particularly vulnerable to imbalances in the cell  or energy metabolism caused by either cytoplasmic or mitochondrial class I and class II ARSs (Najmabadi et al., 2011). As reviewed by Valnegri, Sala, and Passafaro (2012), more than 50% of ID related proteins are enriched in synaptic compartments and may be involved in synaptic plasticity, synapse formation and cytoskeleton rearrangement (see also Ropers &amp: Hamel, 2005). Histological data, which also show a correlation between the severity of ID and the severity of dendritic  <span style="color:red" title="HPO: HP:0000925
CONF: 0.81842345
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >spine malformations</span>  (Valnegri et al., 2012), further corroborate the hypothesis that some features of ID are caused by alterations in synaptic functions. Thus, for further experiments aimed at investigating a possible effect on  <span style="color:red" title="HPO: HP:0012757
CONF: 0.82205933
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >synapse morphology neuronal</span>  cell lines or primary neurons might be advantageous.<br>Interestingly, SARS and WARS2 also seem to interact directly in some cases, as an analysis using the STRING (http://string-db.org, (Szklarczyk et al., 2015), last accessed 07/2016) database showed. Still, in which way this could be relevant in the context of the human  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >brain</span>  remains to be investigated.<br>With , , and the previously implicated , a non catalytic component of the multi tRNA synthetase complex and modulator of aminoacylation activity of cytoplasmic arginyl tRNA synthase RRS (Iqbal et al., 2016) there are now at least three genes involved in tRNA aminoacylation that play a role in the pathology of hereditary  <span style="color:red" title="HPO: HP:0100543
CONF: 0.9595157
PAT: None
NEG: False
REL: False
SECTION: 4 DISCUSSION" >cognitive</span>  disorders. In this context, it is worthwhile mentioning that various tRNA modifiers were already found to play a role in the etiology of ID as well (see Balke, Kuss, &amp: Muller, 2015 for review). It therefore seems that tRNA modification and  aminoacylation are emerging to be among the more prominent pathways contributing to the molecular background of human  <span style="color:red" title="HPO: HP:0100543
CONF: 0.95951575
PAT: None
NEG: False
REL: False
SECTION:  ACKNOWLEDGMENTS" >cognitive</span>  disorders and in conclusion, we would like to present the notion that not only a full but also a fully functional complement of tRNAs is vital for the development and maintenance of higher  <span style="color:red" title="HPO: HP:0012443
CONF: 0.82531583
PAT: None
NEG: False
REL: False
SECTION:  ACKNOWLEDGMENTS" >brain</span>  functions.<br><h2> ACKNOWLEDGMENTS</h2> <br> We are very grateful to the affected individuals and their family for their participation in the study. We thank Susanne Freier and Daniel Mehnert for expert technical assistance. We also thank Robert Weissmann for excellent bioinformatics analyses. H. H. R., L. M., H. N., H. H., and K. K. participated in the GENCODYS Consortium.<br><h2> DISCLOSURE STATEMENT</h2> <br> The authors declare no conflict of interest.<br><h2><br> <h5>Supporting Information</h2> <br>Please note: The publisher is not responsible for the content or functionality of any supporting information supplied by the authors. Any queries (other than missing content) should be directed to the corresponding author for the article.<br><h2><br> REFERENCES<br> </h2> <br><h2>Citing Literature</h2> <h3>Number of times cited according to CrossRef: 31</h3> <br>Volume38, Issue6<br> June 2017<br> Pages 621-636<br>Close Figure ViewerReturn to FigurePrevious FigureNext Figure<h5>Caption<br> Download PDF<br><h2><br>Additional links<br></h2> <br><h3>About Wiley Online Library</h3> <br><h3>Help &amp: Support</h3> <br><h3>Opportunities</h3> <br><h3>Connect with Wiley</h3> <br>Copyright (c) 1999-2020 John Wiley &amp: Sons, Inc. All rights reserved<br>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<br><h5><br> <h5><br><h2>Log in to Wiley Online Library</h2> <br>Email or Customer ID<br>Password<br>Forgot password?<br>NEW USER &gt:<br>INSTITUTIONAL LOGIN &gt:<br><h5><br> <h5><br><h2>Change Password</h2> <br>Old Password<br>New Password<br>Too Short<br> Weak<br> Medium<br> Strong<br> Very Strong<br> Too Long<br><h2>Password Changed Successfully</h2> <br> Your password has been changed<br><h5><br><h2>Create a new account</h2> <br>Email or Customer ID<br>Returning user<br><h5><br> <br> <h5><br><h2>Forgot your password?</h2> <br> Enter your email address below.<br>Email or Customer ID<br>Please check your email for instructions on resetting your password.<br>If you do not receive an email within 10 minutes, your email address may not be registered,<br>and you may need to create a new Wiley Online Library account.<br><h2>Request Username</h2> <br> <br> Can't sign in? Forgot your username?<br> <br> Enter your email address below and we will send you your username<br>Email or Customer ID<br>Close<br>If the address matches an existing account you will receive an email with instructions to retrieve your username<br>